Combined from Debian patches:
Channel_dep_eps
Decomposition_eps
Includes_eps
Includes_fig
Lifting_dep_eps
Waili_eps
Wavelet_dep_eps
--- /dev/null
+++ waili-19990723/man/Channel_dep.eps
@@ -0,0 +1,202 @@
+%!PS-Adobe-2.0
+%%Title: Channel_dep.fig
+%%Creator: fig2dev Version 3.2 Patchlevel 3b
+%%CreationDate: Thu May 17 12:30:00 2001
+%%For: debian@etabeta (A C G Mennucc,,,)
+%%Orientation: Portrait
+%%Pages: 1
+%%BoundingBox: 0 0 612 792
+%%BeginSetup
+%%IncludeFeature: *PageSize Letter
+%%EndSetup
+%%Magnification: 1.0000
+%%EndComments
+%
+%   
+%    Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium 
+%   
+%    This program is free software; you can redistribute it and/or modify 
+%    it under the terms of the GNU General Public License as published by 
+%    the Free Software Foundation; either version 2 of the License, or 
+%    (at your option) any later version. 
+%   
+%    This program is distributed in the hope that it will be useful, 
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of 
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+%    GNU General Public License for more details. 
+%   
+%    You should have received a copy of the GNU General Public License 
+%    along with this program; if not, write to the Free Software 
+%    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+%   
+%
+/$F2psDict 200 dict def
+$F2psDict begin
+$F2psDict /mtrx matrix put
+/col-1 {0 setgray} bind def
+/col0 {0.000 0.000 0.000 srgb} bind def
+/col1 {0.000 0.000 1.000 srgb} bind def
+/col2 {0.000 1.000 0.000 srgb} bind def
+/col3 {0.000 1.000 1.000 srgb} bind def
+/col4 {1.000 0.000 0.000 srgb} bind def
+/col5 {1.000 0.000 1.000 srgb} bind def
+/col6 {1.000 1.000 0.000 srgb} bind def
+/col7 {1.000 1.000 1.000 srgb} bind def
+/col8 {0.000 0.000 0.560 srgb} bind def
+/col9 {0.000 0.000 0.690 srgb} bind def
+/col10 {0.000 0.000 0.820 srgb} bind def
+/col11 {0.530 0.810 1.000 srgb} bind def
+/col12 {0.000 0.560 0.000 srgb} bind def
+/col13 {0.000 0.690 0.000 srgb} bind def
+/col14 {0.000 0.820 0.000 srgb} bind def
+/col15 {0.000 0.560 0.560 srgb} bind def
+/col16 {0.000 0.690 0.690 srgb} bind def
+/col17 {0.000 0.820 0.820 srgb} bind def
+/col18 {0.560 0.000 0.000 srgb} bind def
+/col19 {0.690 0.000 0.000 srgb} bind def
+/col20 {0.820 0.000 0.000 srgb} bind def
+/col21 {0.560 0.000 0.560 srgb} bind def
+/col22 {0.690 0.000 0.690 srgb} bind def
+/col23 {0.820 0.000 0.820 srgb} bind def
+/col24 {0.500 0.190 0.000 srgb} bind def
+/col25 {0.630 0.250 0.000 srgb} bind def
+/col26 {0.750 0.380 0.000 srgb} bind def
+/col27 {1.000 0.500 0.500 srgb} bind def
+/col28 {1.000 0.630 0.630 srgb} bind def
+/col29 {1.000 0.750 0.750 srgb} bind def
+/col30 {1.000 0.880 0.880 srgb} bind def
+/col31 {1.000 0.840 0.000 srgb} bind def
+
+end
+save
+newpath 0 792 moveto 0 0 lineto 612 0 lineto 612 792 lineto closepath clip newpath
+142.0 508.5 translate
+1 -1 scale
+
+/cp {closepath} bind def
+/ef {eofill} bind def
+/gr {grestore} bind def
+/gs {gsave} bind def
+/sa {save} bind def
+/rs {restore} bind def
+/l {lineto} bind def
+/m {moveto} bind def
+/rm {rmoveto} bind def
+/n {newpath} bind def
+/s {stroke} bind def
+/sh {show} bind def
+/slc {setlinecap} bind def
+/slj {setlinejoin} bind def
+/slw {setlinewidth} bind def
+/srgb {setrgbcolor} bind def
+/rot {rotate} bind def
+/sc {scale} bind def
+/sd {setdash} bind def
+/ff {findfont} bind def
+/sf {setfont} bind def
+/scf {scalefont} bind def
+/sw {stringwidth} bind def
+/tr {translate} bind def
+/tnt {dup dup currentrgbcolor
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add srgb}
+  bind def
+/shd {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
+  4 -2 roll mul srgb} bind def
+/$F2psBegin {$F2psDict begin /$F2psEnteredState save def} def
+/$F2psEnd {$F2psEnteredState restore end} def
+
+$F2psBegin
+%%Page: 1 1
+10 setmiterlimit
+ 0.06000 0.06000 sc
+%%Page: 1 1
+%
+% Fig objects follow
+%
+% Polyline
+7.500 slw
+n 2625 1650 m 3675 1650 l 3675 2100 l 2625 2100 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+2775 1950 m
+gs 1 -1 sc (LChannel) col-1 sh gr
+% Polyline
+n 525 1650 m 1575 1650 l 1575 2100 l 525 2100 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+600 1950 m
+gs 1 -1 sc (NTChannel) col-1 sh gr
+% Polyline
+n 1575 450 m 2625 450 l 2625 900 l 1575 900 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+1800 750 m
+gs 1 -1 sc (Channel) col-1 sh gr
+% Polyline
+gs  clippath
+1846 2831 m 1877 2883 l 2007 2807 l 1889 2842 l 1977 2755 l cp
+eoclip
+n 3150 2100 m
+ 1875 2850 l gs col-1 s gr gr
+
+% arrowhead
+n 1977 2755 m 1889 2842 l 2007 2807 l 1977 2755 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+4422 2883 m 4453 2831 l 4322 2755 l 4411 2842 l 4292 2807 l cp
+eoclip
+n 3150 2100 m
+ 4425 2850 l gs col-1 s gr gr
+
+% arrowhead
+n 4292 2807 m 4411 2842 l 4322 2755 l 4292 2807 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+1020 1634 m 1055 1683 l 1178 1595 l 1063 1641 l 1143 1546 l cp
+eoclip
+n 2100 900 m
+ 1050 1650 l gs col-1 s gr gr
+
+% arrowhead
+n 1143 1546 m 1063 1641 l 1178 1595 l 1143 1546 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+3144 1683 m 3179 1634 l 3056 1546 l 3137 1641 l 3021 1595 l cp
+eoclip
+n 2100 900 m
+ 3150 1650 l gs col-1 s gr gr
+
+% arrowhead
+n 3021 1595 m 3137 1641 l 3056 1546 l 3021 1595 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+n 2625 2850 m 3675 2850 l 3675 3300 l 2625 3300 l
+ cp gs col-1 s gr 
+% Polyline
+n 1350 2850 m 2400 2850 l 2400 3300 l 1350 3300 l
+ cp gs col-1 s gr 
+% Polyline
+n 3900 2850 m 4950 2850 l 4950 3300 l 3900 3300 l
+ cp gs col-1 s gr 
+% Polyline
+gs  clippath
+3120 2865 m 3180 2865 l 3180 2714 l 3150 2834 l 3120 2714 l cp
+eoclip
+n 3150 2100 m
+ 3150 2850 l gs col-1 s gr gr
+
+% arrowhead
+n 3120 2714 m 3150 2834 l 3180 2714 l 3120 2714 l  cp gs 0.00 setgray ef gr  col-1 s
+/Times-Roman ff 180.00 scf sf
+2700 3150 m
+gs 1 -1 sc (LChannelC) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+3975 3150 m
+gs 1 -1 sc (LChannelR) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+1388 3150 m
+gs 1 -1 sc (LChannelCR) col-1 sh gr
+$F2psEnd
+rs
+showpage
--- /dev/null
+++ waili-19990723/man/Decomposition.eps
@@ -0,0 +1,200 @@
+%!PS-Adobe-2.0
+%%Title: Decomposition.fig
+%%Creator: fig2dev Version 3.2 Patchlevel 3b
+%%CreationDate: Thu May 17 12:30:01 2001
+%%For: debian@etabeta (A C G Mennucc,,,)
+%%Orientation: Landscape
+%%Pages: 1
+%%BoundingBox: 0 0 612 792
+%%BeginSetup
+%%IncludeFeature: *PageSize Letter
+%%EndSetup
+%%Magnification: 1.0000
+%%EndComments
+%
+%   
+%    Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium 
+%   
+%    This program is free software; you can redistribute it and/or modify 
+%    it under the terms of the GNU General Public License as published by 
+%    the Free Software Foundation; either version 2 of the License, or 
+%    (at your option) any later version. 
+%   
+%    This program is distributed in the hope that it will be useful, 
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of 
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+%    GNU General Public License for more details. 
+%   
+%    You should have received a copy of the GNU General Public License 
+%    along with this program; if not, write to the Free Software 
+%    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+%   
+%
+/$F2psDict 200 dict def
+$F2psDict begin
+$F2psDict /mtrx matrix put
+/col-1 {0 setgray} bind def
+/col0 {0.000 0.000 0.000 srgb} bind def
+/col1 {0.000 0.000 1.000 srgb} bind def
+/col2 {0.000 1.000 0.000 srgb} bind def
+/col3 {0.000 1.000 1.000 srgb} bind def
+/col4 {1.000 0.000 0.000 srgb} bind def
+/col5 {1.000 0.000 1.000 srgb} bind def
+/col6 {1.000 1.000 0.000 srgb} bind def
+/col7 {1.000 1.000 1.000 srgb} bind def
+/col8 {0.000 0.000 0.560 srgb} bind def
+/col9 {0.000 0.000 0.690 srgb} bind def
+/col10 {0.000 0.000 0.820 srgb} bind def
+/col11 {0.530 0.810 1.000 srgb} bind def
+/col12 {0.000 0.560 0.000 srgb} bind def
+/col13 {0.000 0.690 0.000 srgb} bind def
+/col14 {0.000 0.820 0.000 srgb} bind def
+/col15 {0.000 0.560 0.560 srgb} bind def
+/col16 {0.000 0.690 0.690 srgb} bind def
+/col17 {0.000 0.820 0.820 srgb} bind def
+/col18 {0.560 0.000 0.000 srgb} bind def
+/col19 {0.690 0.000 0.000 srgb} bind def
+/col20 {0.820 0.000 0.000 srgb} bind def
+/col21 {0.560 0.000 0.560 srgb} bind def
+/col22 {0.690 0.000 0.690 srgb} bind def
+/col23 {0.820 0.000 0.820 srgb} bind def
+/col24 {0.500 0.190 0.000 srgb} bind def
+/col25 {0.630 0.250 0.000 srgb} bind def
+/col26 {0.750 0.380 0.000 srgb} bind def
+/col27 {1.000 0.500 0.500 srgb} bind def
+/col28 {1.000 0.630 0.630 srgb} bind def
+/col29 {1.000 0.750 0.750 srgb} bind def
+/col30 {1.000 0.880 0.880 srgb} bind def
+/col31 {1.000 0.840 0.000 srgb} bind def
+
+end
+save
+newpath 0 792 moveto 0 0 lineto 612 0 lineto 612 792 lineto closepath clip newpath
+135.0 -126.0 translate
+ 90 rotate
+1 -1 scale
+
+/cp {closepath} bind def
+/ef {eofill} bind def
+/gr {grestore} bind def
+/gs {gsave} bind def
+/sa {save} bind def
+/rs {restore} bind def
+/l {lineto} bind def
+/m {moveto} bind def
+/rm {rmoveto} bind def
+/n {newpath} bind def
+/s {stroke} bind def
+/sh {show} bind def
+/slc {setlinecap} bind def
+/slj {setlinejoin} bind def
+/slw {setlinewidth} bind def
+/srgb {setrgbcolor} bind def
+/rot {rotate} bind def
+/sc {scale} bind def
+/sd {setdash} bind def
+/ff {findfont} bind def
+/sf {setfont} bind def
+/scf {scalefont} bind def
+/sw {stringwidth} bind def
+/tr {translate} bind def
+/tnt {dup dup currentrgbcolor
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add srgb}
+  bind def
+/shd {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
+  4 -2 roll mul srgb} bind def
+/$F2psBegin {$F2psDict begin /$F2psEnteredState save def} def
+/$F2psEnd {$F2psEnteredState restore end} def
+
+$F2psBegin
+%%Page: 1 1
+10 setmiterlimit
+ 0.06000 0.06000 sc
+%%Page: 1 1
+%
+% Fig objects follow
+%
+% Polyline
+7.500 slw
+n 6675 2250 m 8475 2250 l 8475 2850 l 6675 2850 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+6885 2647 m
+gs 1 -1 sc (NTChannel) col0 sh gr
+% Polyline
+n 6375 2250 m 6675 2250 l 6675 2850 l 6375 2850 l
+ cp gs col0 s gr 
+/Times-Roman ff 270.00 scf sf
+6427 2647 m
+gs 1 -1 sc (H) col0 sh gr
+% Polyline
+n 6375 1500 m 6675 1500 l 6675 2100 l 6375 2100 l
+ cp gs col0 s gr 
+/Times-Roman ff 270.00 scf sf
+6442 1897 m
+gs 1 -1 sc (L) col0 sh gr
+% Polyline
+n 6675 1500 m 8475 1500 l 8475 2100 l 6675 2100 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+6885 1897 m
+gs 1 -1 sc (NTChannel) col0 sh gr
+% Polyline
+n 6225 900 m 8625 900 l 8625 3000 l 6225 3000 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+6735 1297 m
+gs 1 -1 sc (LChannelC) col0 sh gr
+% Polyline
+n 5700 900 m 6225 900 l 6225 3000 l 5700 3000 l
+ cp gs col0 s gr 
+/Times-Roman ff 270.00 scf sf
+5775 2100 m
+gs 1 -1 sc (LL) col0 sh gr
+% Polyline
+n 6225 3150 m 8625 3150 l 8625 5250 l 6225 5250 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+6735 4297 m
+gs 1 -1 sc (NTChannel) col0 sh gr
+% Polyline
+n 5700 3150 m 6225 3150 l 6225 5250 l 5700 5250 l
+ cp gs col0 s gr 
+/Times-Roman ff 270.00 scf sf
+5782 4297 m
+gs 1 -1 sc (HL) col0 sh gr
+% Polyline
+n 9300 900 m 11700 900 l 11700 3000 l 9300 3000 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+9810 2047 m
+gs 1 -1 sc (NTChannel) col0 sh gr
+% Polyline
+n 8775 900 m 9300 900 l 9300 3000 l 8775 3000 l
+ cp gs col0 s gr 
+/Times-Roman ff 270.00 scf sf
+8857 2047 m
+gs 1 -1 sc (LH) col0 sh gr
+% Polyline
+n 9300 3150 m 11700 3150 l 11700 5250 l 9300 5250 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+9810 4297 m
+gs 1 -1 sc (NTChannel) col0 sh gr
+% Polyline
+n 8775 3150 m 9300 3150 l 9300 5250 l 8775 5250 l
+ cp gs col0 s gr 
+/Times-Roman ff 270.00 scf sf
+8842 4297 m
+gs 1 -1 sc (HH) col0 sh gr
+% Polyline
+n 5550 300 m 11850 300 l 11850 5400 l 5550 5400 l
+ cp gs col0 s gr 
+/Times-BoldItalic ff 270.00 scf sf
+7920 697 m
+gs 1 -1 sc (LChannelCR) col0 sh gr
+$F2psEnd
+rs
+showpage
Index: waili-19990723/man/Includes.eps
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ waili-19990723/man/Includes.eps	2014-01-13 15:29:41.000000000 +0100
@@ -0,0 +1,1018 @@
+%!PS
+%%Title: Includes.fig
+%%Creator: fig2dev Version 3.2 Patchlevel 5c
+%%CreationDate: Mon Jan 13 15:29:41 2014
+%%Orientation: Portrait
+%%Pages: 2
+%%BoundingBox: 0 0 595 842
+%%DocumentPaperSizes: a4
+%%BeginSetup
+[{
+%%BeginFeature: *PageRegion A4
+<</PageSize [595 842]>> setpagedevice
+%%EndFeature
+} stopped cleartomark
+%%EndSetup
+%Magnification: 1.0000
+%%EndComments
+%
+%   
+%    Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium 
+%   
+%    This program is free software; you can redistribute it and/or modify 
+%    it under the terms of the GNU General Public License as published by 
+%    the Free Software Foundation; either version 2 of the License, or 
+%    (at your option) any later version. 
+%   
+%    This program is distributed in the hope that it will be useful, 
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of 
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+%    GNU General Public License for more details. 
+%   
+%    You should have received a copy of the GNU General Public License 
+%    along with this program; if not, write to the Free Software 
+%    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+%   
+%
+%%BeginProlog
+/$F2psDict 200 dict def
+$F2psDict begin
+$F2psDict /mtrx matrix put
+/col-1 {0 setgray} bind def
+/col0 {0.000 0.000 0.000 srgb} bind def
+/col1 {0.000 0.000 1.000 srgb} bind def
+/col2 {0.000 1.000 0.000 srgb} bind def
+/col3 {0.000 1.000 1.000 srgb} bind def
+/col4 {1.000 0.000 0.000 srgb} bind def
+/col5 {1.000 0.000 1.000 srgb} bind def
+/col6 {1.000 1.000 0.000 srgb} bind def
+/col7 {1.000 1.000 1.000 srgb} bind def
+/col8 {0.000 0.000 0.560 srgb} bind def
+/col9 {0.000 0.000 0.690 srgb} bind def
+/col10 {0.000 0.000 0.820 srgb} bind def
+/col11 {0.530 0.810 1.000 srgb} bind def
+/col12 {0.000 0.560 0.000 srgb} bind def
+/col13 {0.000 0.690 0.000 srgb} bind def
+/col14 {0.000 0.820 0.000 srgb} bind def
+/col15 {0.000 0.560 0.560 srgb} bind def
+/col16 {0.000 0.690 0.690 srgb} bind def
+/col17 {0.000 0.820 0.820 srgb} bind def
+/col18 {0.560 0.000 0.000 srgb} bind def
+/col19 {0.690 0.000 0.000 srgb} bind def
+/col20 {0.820 0.000 0.000 srgb} bind def
+/col21 {0.560 0.000 0.560 srgb} bind def
+/col22 {0.690 0.000 0.690 srgb} bind def
+/col23 {0.820 0.000 0.820 srgb} bind def
+/col24 {0.500 0.190 0.000 srgb} bind def
+/col25 {0.630 0.250 0.000 srgb} bind def
+/col26 {0.750 0.380 0.000 srgb} bind def
+/col27 {1.000 0.500 0.500 srgb} bind def
+/col28 {1.000 0.630 0.630 srgb} bind def
+/col29 {1.000 0.750 0.750 srgb} bind def
+/col30 {1.000 0.880 0.880 srgb} bind def
+/col31 {1.000 0.840 0.000 srgb} bind def
+
+end
+
+/cp {closepath} bind def
+/ef {eofill} bind def
+/gr {grestore} bind def
+/gs {gsave} bind def
+/sa {save} bind def
+/rs {restore} bind def
+/l {lineto} bind def
+/m {moveto} bind def
+/rm {rmoveto} bind def
+/n {newpath} bind def
+/s {stroke} bind def
+/sh {show} bind def
+/slc {setlinecap} bind def
+/slj {setlinejoin} bind def
+/slw {setlinewidth} bind def
+/srgb {setrgbcolor} bind def
+/rot {rotate} bind def
+/sc {scale} bind def
+/sd {setdash} bind def
+/ff {findfont} bind def
+/sf {setfont} bind def
+/scf {scalefont} bind def
+/sw {stringwidth} bind def
+/tr {translate} bind def
+/tnt {dup dup currentrgbcolor
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add srgb}
+  bind def
+/shd {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
+  4 -2 roll mul srgb} bind def
+/$F2psBegin {$F2psDict begin /$F2psEnteredState save def} def
+/$F2psEnd {$F2psEnteredState restore end} def
+
+/pageheader {
+save
+newpath 0 842 moveto 0 0 lineto 595 0 lineto 595 842 lineto closepath clip newpath
+-26.5 803.5 translate
+$F2psBegin
+10 setmiterlimit
+0 slj 0 slc
+ 0.06000 0.06000 sc
+} bind def
+/pagefooter {
+$F2psEnd
+restore
+} bind def
+%%EndProlog
+%%Page: 1 1
+%%BeginPageSetup
+pageheader
+1 -1 scale
+%%EndPageSetup
+%
+% Fig objects follow
+%
+% 
+% here starts figure with depth 0
+% Polyline
+0 slj
+0 slc
+7.500 slw
+n 9375 6150 m 10425 6150 l 10425 6600 l 9375 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 9300 6150 m 10500 6150 l 10500 6600 l 9300 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 7575 6150 m 8625 6150 l 8625 6600 l 7575 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 7500 6150 m 8700 6150 l 8700 6600 l 7500 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 7575 4950 m 8625 4950 l 8625 5400 l 7575 5400 l
+ cp gs col-1 s gr 
+% Polyline
+n 7500 4950 m 8700 4950 l 8700 5400 l 7500 5400 l
+ cp gs col-1 s gr 
+% Polyline
+n 5775 6150 m 6825 6150 l 6825 6600 l 5775 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 5700 6150 m 6900 6150 l 6900 6600 l 5700 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 7500 7350 m 8700 7350 l 8700 7800 l 7500 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 3975 4950 m 5025 4950 l 5025 5400 l 3975 5400 l
+ cp gs col-1 s gr 
+% Polyline
+n 3900 4950 m 5100 4950 l 5100 5400 l 3900 5400 l
+ cp gs col-1 s gr 
+% Polyline
+n 3900 6150 m 5100 6150 l 5100 6600 l 3900 6600 l
+ cp gs col-1 s gr 
+% Polyline
+n 3975 7350 m 5025 7350 l 5025 7800 l 3975 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 3900 7350 m 5100 7350 l 5100 7800 l 3900 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 5775 7350 m 6825 7350 l 6825 7800 l 5775 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 5700 7350 m 6900 7350 l 6900 7800 l 5700 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 9375 7350 m 10425 7350 l 10425 7800 l 9375 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 9300 7350 m 10500 7350 l 10500 7800 l 9300 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 2175 7350 m 3225 7350 l 3225 7800 l 2175 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 2100 7350 m 3300 7350 l 3300 7800 l 2100 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 2100 4950 m 3300 4950 l 3300 5400 l 2100 5400 l
+ cp gs col-1 s gr 
+% Polyline
+n 2175 4950 m 3225 4950 l 3225 5400 l 2175 5400 l
+ cp gs col-1 s gr 
+% Polyline
+n 4875 3750 m 5925 3750 l 5925 4200 l 4875 4200 l
+ cp gs col-1 s gr 
+% Polyline
+n 4800 3750 m 6000 3750 l 6000 4200 l 4800 4200 l
+ cp gs col-1 s gr 
+% Polyline
+n 300 7350 m 1500 7350 l 1500 7800 l 300 7800 l
+ cp gs col-1 s gr 
+% Polyline
+n 4800 8550 m 6000 8550 l 6000 9000 l 4800 9000 l
+ cp gs col-1 s gr 
+% Polyline
+n 4800 9750 m 6000 9750 l 6000 10200 l 4800 10200 l
+ cp gs col-1 s gr 
+% Polyline
+n 2625 2550 m 3675 2550 l 3675 3000 l 2625 3000 l
+ cp gs col-1 s gr 
+% Polyline
+n 2550 2550 m 3750 2550 l 3750 3000 l 2550 3000 l
+ cp gs col-1 s gr 
+% Polyline
+gs  clippath
+2670 7214 m 2670 7365 l 2730 7365 l 2730 7214 l 2730 7214 l 2700 7334 l 2670 7214 l cp
+eoclip
+n 2700 5400 m
+ 2700 7350 l gs col-1 s gr gr
+
+% arrowhead
+n 2670 7214 m 2700 7334 l 2730 7214 l 2670 7214 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+ [60] 0 sd
+gs  clippath
+4470 6013 m 4470 6165 l 4530 6165 l 4530 6013 l 4530 6013 l 4500 6133 l 4470 6013 l cp
+eoclip
+n 4500 5400 m
+ 4500 6150 l gs col-1 s gr gr
+ [] 0 sd
+% arrowhead
+n 4470 6013 m 4500 6133 l 4530 6013 l  col-1 s
+% Polyline
+ [60] 0 sd
+gs  clippath
+8070 6013 m 8070 6165 l 8130 6165 l 8130 6013 l 8130 6013 l 8100 6133 l 8070 6013 l cp
+eoclip
+n 8100 5400 m
+ 8100 6150 l gs col-1 s gr gr
+ [] 0 sd
+% arrowhead
+n 8070 6013 m 8100 6133 l 8130 6013 l  col-1 s
+% Polyline
+gs  clippath
+8070 7213 m 8070 7365 l 8130 7365 l 8130 7213 l 8130 7213 l 8100 7333 l 8070 7213 l cp
+eoclip
+n 8100 6600 m
+ 8100 7350 l gs col-1 s gr gr
+
+% arrowhead
+n 8070 7213 m 8100 7333 l 8130 7213 l  col-1 s
+% Polyline
+gs  clippath
+5370 9614 m 5370 9765 l 5430 9765 l 5430 9614 l 5430 9614 l 5400 9734 l 5370 9614 l cp
+eoclip
+n 5400 9000 m
+ 5400 9750 l gs col-1 s gr gr
+
+% arrowhead
+n 5370 9614 m 5400 9734 l 5430 9614 l 5370 9614 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+8162 7241 m 8229 7376 l 8283 7350 l 8216 7214 l 8216 7214 l 8243 7335 l 8162 7241 l cp
+eoclip
+n 8250 5400 m 8249 5400 l 8248 5400 l 8247 5400 l 8245 5400 l 8244 5401 l
+ 8242 5401 l 8240 5401 l 8238 5402 l 8236 5403 l 8234 5404 l
+ 8232 5405 l 8230 5406 l 8228 5408 l 8227 5409 l 8225 5412 l
+ 8224 5414 l 8224 5417 l 8224 5420 l 8225 5424 l 8226 5429 l
+ 8228 5434 l 8231 5439 l 8235 5446 l 8239 5453 l 8245 5461 l
+ 8252 5471 l 8261 5481 l 8270 5492 l 8282 5505 l 8295 5519 l
+ 8309 5534 l 8325 5550 l 8349 5574 l 8375 5598 l 8400 5620 l
+ 8425 5641 l 8449 5660 l 8472 5676 l 8493 5689 l 8513 5700 l
+ 8531 5709 l 8547 5716 l 8563 5722 l 8579 5727 l 8594 5731 l
+ 8609 5736 l 8624 5742 l 8640 5749 l 8656 5758 l 8674 5769 l
+ 8693 5784 l 8713 5802 l 8735 5824 l 8758 5851 l 8781 5882 l
+ 8805 5918 l 8828 5958 l 8850 6000 l 8869 6044 l 8884 6086 l
+ 8897 6127 l 8907 6164 l 8916 6197 l 8922 6226 l 8927 6251 l
+ 8931 6273 l 8933 6291 l 8935 6306 l 8937 6320 l 8937 6332 l
+ 8938 6344 l 8937 6355 l 8937 6367 l 8935 6381 l 8933 6396 l
+ 8931 6414 l 8927 6435 l 8922 6459 l 8916 6487 l 8907 6519 l
+ 8897 6555 l 8884 6594 l 8869 6634 l 8850 6675 l 8827 6717 l
+ 8801 6756 l 8775 6790 l 8750 6818 l 8726 6841 l 8703 6858 l
+ 8682 6872 l 8662 6881 l 8644 6887 l 8626 6891 l 8610 6893 l
+ 8594 6894 l 8578 6895 l 8561 6896 l 8543 6899 l 8525 6903 l
+ 8505 6910 l 8483 6920 l 8459 6933 l 8434 6950 l 8407 6971 l
+ 8379 6995 l 8351 7022 l 8325 7050 l 8297 7083 l 8275 7114 l
+ 8257 7143 l 8244 7168 l 8235 7191 l 8229 7213 l 8225 7232 l
+ 8224 7250 l 8225 7267 l 8227 7282 l 8230 7296 l 8233 7309 l
+ 8237 7320 l 8241 7330 l 8244 7338 l
+ 8250 7350 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 8162 7241 m 8243 7335 l 8216 7214 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+8067 7273 m 7925 7327 l 7946 7383 l 8088 7329 l 8088 7329 l 7966 7344 l 8067 7273 l cp
+eoclip
+n 6300 6600 m 6299 6600 l 6298 6600 l 6296 6600 l 6294 6599 l 6291 6599 l
+ 6287 6599 l 6283 6598 l 6279 6598 l 6273 6597 l 6267 6597 l
+ 6261 6596 l 6254 6595 l 6247 6595 l 6239 6594 l 6231 6593 l
+ 6223 6593 l 6215 6592 l 6208 6592 l 6200 6591 l 6192 6591 l
+ 6185 6591 l 6178 6591 l 6172 6591 l 6167 6591 l 6162 6591 l
+ 6158 6592 l 6155 6593 l 6153 6594 l 6152 6595 l 6152 6597 l
+ 6153 6599 l 6156 6601 l 6161 6604 l 6166 6607 l 6174 6610 l
+ 6184 6614 l 6195 6619 l 6209 6624 l 6225 6629 l 6244 6635 l
+ 6265 6642 l 6288 6649 l 6315 6657 l 6343 6666 l 6375 6675 l
+ 6418 6687 l 6464 6700 l 6512 6713 l 6560 6725 l 6607 6737 l
+ 6654 6749 l 6700 6760 l 6744 6770 l 6787 6780 l 6827 6789 l
+ 6866 6797 l 6903 6804 l 6938 6811 l 6971 6817 l 7004 6823 l
+ 7035 6829 l 7065 6834 l 7095 6839 l 7125 6844 l 7155 6849 l
+ 7185 6854 l 7215 6859 l 7246 6865 l 7279 6871 l 7312 6878 l
+ 7347 6886 l 7384 6894 l 7423 6903 l 7463 6914 l 7506 6925 l
+ 7550 6937 l 7596 6951 l 7643 6965 l 7690 6981 l 7738 6997 l
+ 7786 7015 l 7832 7032 l 7875 7050 l 7914 7068 l 7950 7085 l
+ 7981 7101 l 8007 7117 l 8030 7132 l 8049 7147 l 8065 7160 l
+ 8077 7173 l 8086 7186 l 8092 7197 l 8096 7208 l 8098 7219 l
+ 8098 7229 l 8096 7239 l 8092 7248 l 8087 7257 l 8081 7266 l
+ 8073 7274 l 8065 7282 l 8056 7290 l 8046 7297 l 8036 7304 l
+ 8026 7310 l 8016 7316 l 8006 7322 l 7997 7327 l 7988 7332 l
+ 7980 7336 l 7973 7339 l 7967 7342 l 7962 7345 l 7958 7347 l
+
+ 7950 7350 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 8067 7273 m 7966 7344 l 8088 7329 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+8260 7352 m 8408 7382 l 8420 7323 l 8272 7294 l 8272 7294 l 8384 7347 l 8260 7352 l cp
+eoclip
+n 9900 6600 m 9901 6600 l 9902 6600 l 9904 6599 l 9906 6599 l 9908 6598 l
+ 9911 6598 l 9915 6597 l 9919 6596 l 9924 6596 l 9929 6595 l
+ 9934 6594 l 9941 6592 l 9947 6591 l 9954 6590 l 9960 6589 l
+ 9967 6588 l 9974 6587 l 9981 6586 l 9988 6585 l 9995 6584 l
+ 10001 6584 l 10007 6583 l 10012 6583 l 10017 6583 l 10021 6583 l
+ 10024 6583 l 10027 6584 l 10028 6584 l 10029 6586 l 10029 6587 l
+ 10027 6589 l 10024 6591 l 10020 6594 l 10015 6598 l 10008 6601 l
+ 9999 6606 l 9988 6611 l 9976 6616 l 9961 6622 l 9944 6629 l
+ 9925 6637 l 9904 6645 l 9880 6655 l 9854 6664 l 9825 6675 l
+ 9784 6690 l 9740 6706 l 9694 6722 l 9649 6738 l 9603 6753 l
+ 9559 6768 l 9516 6782 l 9475 6795 l 9435 6807 l 9398 6819 l
+ 9362 6829 l 9328 6839 l 9296 6848 l 9265 6857 l 9235 6865 l
+ 9206 6873 l 9178 6880 l 9150 6888 l 9122 6895 l 9094 6903 l
+ 9065 6910 l 9035 6919 l 9004 6928 l 8972 6937 l 8938 6948 l
+ 8902 6959 l 8865 6971 l 8825 6985 l 8784 6999 l 8741 7015 l
+ 8697 7032 l 8651 7049 l 8606 7068 l 8560 7087 l 8516 7106 l
+ 8475 7125 l 8443 7141 l 8413 7156 l 8387 7170 l 8364 7184 l
+ 8344 7197 l 8326 7208 l 8312 7220 l 8300 7230 l 8290 7239 l
+ 8283 7248 l 8277 7257 l 8273 7264 l 8271 7272 l 8271 7278 l
+ 8272 7285 l 8274 7290 l 8277 7296 l 8282 7301 l 8287 7306 l
+ 8293 7311 l 8299 7315 l 8306 7319 l 8314 7323 l 8322 7326 l
+ 8330 7329 l 8338 7332 l 8345 7335 l 8353 7338 l 8360 7340 l
+ 8367 7342 l 8373 7344 l 8379 7345 l 8384 7346 l 8388 7347 l
+ 8392 7348 l 8395 7349 l
+ 8400 7350 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 8260 7352 m 8384 7347 l 8272 7294 l  col-1 s
+% Polyline
+2 slj
+ [60] 0 sd
+gs  clippath
+10037 6120 m 9885 6120 l 9885 6180 l 10037 6180 l 10037 6180 l 9917 6150 l 10037 6120 l cp
+eoclip
+n 8400 5400 m 8399 5400 l 8398 5399 l 8396 5399 l 8394 5399 l 8392 5398 l
+ 8389 5397 l 8385 5396 l 8381 5395 l 8376 5394 l 8371 5393 l
+ 8366 5391 l 8359 5390 l 8353 5388 l 8346 5387 l 8340 5385 l
+ 8333 5383 l 8326 5382 l 8319 5380 l 8312 5379 l 8305 5378 l
+ 8299 5377 l 8293 5376 l 8288 5375 l 8283 5374 l 8279 5374 l
+ 8276 5374 l 8273 5374 l 8272 5375 l 8271 5376 l 8271 5378 l
+ 8273 5379 l 8276 5382 l 8280 5385 l 8285 5388 l 8292 5392 l
+ 8301 5397 l 8312 5402 l 8324 5408 l 8339 5415 l 8356 5423 l
+ 8375 5432 l 8396 5441 l 8420 5452 l 8446 5463 l 8475 5475 l
+ 8514 5491 l 8556 5508 l 8598 5526 l 8642 5543 l 8685 5561 l
+ 8727 5577 l 8768 5593 l 8808 5608 l 8846 5623 l 8883 5637 l
+ 8917 5649 l 8950 5661 l 8982 5673 l 9012 5684 l 9041 5694 l
+ 9069 5703 l 9097 5713 l 9123 5722 l 9150 5731 l 9177 5740 l
+ 9203 5750 l 9231 5759 l 9259 5769 l 9288 5779 l 9318 5790 l
+ 9350 5802 l 9383 5814 l 9417 5828 l 9454 5842 l 9492 5857 l
+ 9532 5873 l 9573 5889 l 9615 5907 l 9658 5925 l 9702 5944 l
+ 9744 5963 l 9786 5982 l 9825 6000 l 9855 6014 l 9882 6028 l
+ 9907 6040 l 9929 6052 l 9948 6063 l 9965 6073 l 9980 6082 l
+ 9992 6090 l 10002 6098 l 10011 6105 l 10018 6111 l 10023 6116 l
+ 10026 6121 l 10028 6126 l 10029 6130 l 10029 6133 l 10027 6136 l
+ 10025 6139 l 10021 6141 l 10017 6143 l 10012 6145 l 10007 6146 l
+ 10001 6148 l 9994 6149 l 9987 6149 l 9980 6150 l 9973 6151 l
+ 9966 6151 l 9959 6151 l 9952 6151 l 9945 6151 l 9938 6151 l
+ 9932 6151 l 9926 6151 l 9921 6151 l 9916 6151 l 9912 6151 l
+ 9909 6150 l 9906 6150 l
+ 9900 6150 l gs col-1 s gr gr
+ [] 0 sd
+% arrowhead
+0 slj
+n 10037 6120 m 9917 6150 l 10037 6180 l  col-1 s
+% Polyline
+2 slj
+ [60] 0 sd
+gs  clippath
+8237 4920 m 8085 4920 l 8085 4980 l 8237 4980 l 8237 4980 l 8117 4950 l 8237 4920 l cp
+eoclip
+n 5625 4200 m 5624 4200 l 5623 4200 l 5621 4199 l 5618 4199 l 5614 4199 l
+ 5610 4198 l 5604 4197 l 5597 4196 l 5590 4195 l 5581 4194 l
+ 5571 4193 l 5560 4191 l 5549 4190 l 5537 4188 l 5524 4187 l
+ 5511 4185 l 5498 4183 l 5484 4182 l 5471 4180 l 5458 4179 l
+ 5445 4178 l 5433 4177 l 5421 4176 l 5411 4175 l 5401 4174 l
+ 5392 4174 l 5385 4174 l 5379 4174 l 5374 4175 l 5372 4176 l
+ 5371 4178 l 5372 4179 l 5375 4182 l 5380 4185 l 5389 4188 l
+ 5399 4192 l 5413 4197 l 5430 4202 l 5450 4208 l 5474 4215 l
+ 5501 4223 l 5533 4232 l 5568 4241 l 5608 4252 l 5652 4263 l
+ 5700 4275 l 5752 4288 l 5806 4301 l 5863 4315 l 5921 4329 l
+ 5980 4343 l 6038 4356 l 6096 4370 l 6153 4383 l 6208 4395 l
+ 6262 4407 l 6315 4419 l 6365 4430 l 6414 4440 l 6461 4450 l
+ 6507 4460 l 6551 4469 l 6593 4478 l 6634 4486 l 6674 4494 l
+ 6713 4502 l 6751 4509 l 6789 4517 l 6826 4524 l 6863 4531 l
+ 6899 4538 l 6936 4546 l 6974 4553 l 7012 4561 l 7051 4569 l
+ 7091 4577 l 7132 4585 l 7174 4594 l 7218 4603 l 7264 4613 l
+ 7311 4624 l 7360 4634 l 7410 4646 l 7463 4658 l 7517 4671 l
+ 7572 4684 l 7629 4697 l 7687 4712 l 7745 4726 l 7804 4741 l
+ 7862 4756 l 7919 4771 l 7973 4786 l 8025 4800 l 8073 4814 l
+ 8117 4827 l 8157 4839 l 8192 4850 l 8224 4861 l 8251 4871 l
+ 8275 4880 l 8295 4888 l 8312 4895 l 8326 4902 l 8336 4908 l
+ 8345 4914 l 8350 4919 l 8353 4923 l 8354 4927 l 8353 4931 l
+ 8351 4934 l 8346 4937 l 8340 4939 l 8333 4942 l 8324 4943 l
+ 8314 4945 l 8304 4946 l 8292 4948 l 8280 4949 l 8267 4949 l
+ 8254 4950 l 8241 4950 l 8227 4951 l 8214 4951 l 8201 4951 l
+ 8188 4951 l 8176 4951 l 8165 4951 l 8154 4951 l 8144 4951 l
+ 8135 4951 l 8128 4951 l 8121 4951 l 8115 4950 l 8111 4950 l
+ 8107 4950 l
+ 8100 4950 l gs col-1 s gr gr
+ [] 0 sd
+% arrowhead
+0 slj
+n 8237 4920 m 8117 4950 l 8237 4980 l  col-1 s
+% Polyline
+2 slj
+ [60] 0 sd
+gs  clippath
+4363 4980 m 4515 4980 l 4515 4920 l 4363 4920 l 4363 4920 l 4483 4950 l 4363 4980 l cp
+eoclip
+n 5325 4200 m 5326 4200 l 5327 4200 l 5329 4200 l 5330 4200 l 5332 4200 l
+ 5334 4201 l 5337 4201 l 5339 4201 l 5342 4202 l 5345 4202 l
+ 5348 4203 l 5351 4204 l 5354 4205 l 5357 4206 l 5360 4208 l
+ 5363 4209 l 5365 4211 l 5366 4214 l 5368 4216 l 5369 4219 l
+ 5369 4222 l 5369 4226 l 5368 4230 l 5366 4235 l 5363 4240 l
+ 5359 4246 l 5355 4252 l 5349 4260 l 5342 4268 l 5333 4277 l
+ 5323 4286 l 5312 4297 l 5299 4309 l 5284 4322 l 5268 4335 l
+ 5250 4350 l 5222 4372 l 5192 4395 l 5162 4417 l 5132 4438 l
+ 5104 4458 l 5077 4476 l 5051 4493 l 5028 4508 l 5006 4522 l
+ 4985 4534 l 4966 4545 l 4947 4556 l 4930 4565 l 4912 4575 l
+ 4895 4585 l 4878 4594 l 4859 4605 l 4840 4616 l 4819 4628 l
+ 4797 4642 l 4774 4657 l 4748 4674 l 4721 4692 l 4693 4712 l
+ 4663 4733 l 4633 4755 l 4603 4778 l 4575 4800 l 4557 4815 l
+ 4541 4828 l 4526 4841 l 4513 4853 l 4502 4864 l 4492 4873 l
+ 4483 4882 l 4476 4890 l 4470 4898 l 4466 4904 l 4462 4910 l
+ 4459 4915 l 4457 4920 l 4456 4924 l 4456 4928 l 4456 4931 l
+ 4457 4934 l 4459 4936 l 4460 4939 l 4463 4941 l 4465 4942 l
+ 4468 4944 l 4471 4945 l 4474 4946 l 4477 4947 l 4480 4948 l
+ 4483 4948 l 4486 4949 l 4488 4949 l 4491 4949 l 4493 4950 l
+ 4495 4950 l
+ 4500 4950 l gs col-1 s gr gr
+ [] 0 sd
+% arrowhead
+0 slj
+n 4363 4980 m 4483 4950 l 4363 4920 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+7765 7271 m 7624 7327 l 7647 7383 l 7787 7327 l 7787 7327 l 7665 7344 l 7765 7271 l cp
+eoclip
+n 4650 5400 m 4649 5401 l 4648 5401 l 4647 5402 l 4646 5404 l 4644 5406 l
+ 4641 5408 l 4639 5411 l 4637 5415 l 4635 5419 l 4634 5423 l
+ 4633 5429 l 4633 5434 l 4634 5441 l 4636 5448 l 4639 5456 l
+ 4644 5465 l 4651 5475 l 4661 5487 l 4672 5500 l 4687 5515 l
+ 4705 5532 l 4725 5550 l 4750 5571 l 4776 5590 l 4801 5607 l
+ 4825 5621 l 4846 5630 l 4865 5636 l 4881 5638 l 4896 5637 l
+ 4908 5634 l 4920 5630 l 4931 5625 l 4943 5621 l 4955 5618 l
+ 4968 5619 l 4984 5623 l 5002 5632 l 5023 5647 l 5048 5670 l
+ 5076 5702 l 5107 5742 l 5141 5792 l 5175 5850 l 5198 5893 l
+ 5219 5938 l 5239 5983 l 5256 6027 l 5270 6069 l 5280 6109 l
+ 5289 6146 l 5294 6181 l 5296 6213 l 5297 6243 l 5295 6271 l
+ 5292 6297 l 5287 6321 l 5281 6344 l 5275 6366 l 5269 6388 l
+ 5263 6409 l 5257 6431 l 5252 6454 l 5249 6478 l 5248 6503 l
+ 5249 6530 l 5253 6559 l 5260 6590 l 5271 6624 l 5286 6660 l
+ 5306 6698 l 5330 6737 l 5359 6779 l 5394 6820 l 5432 6861 l
+ 5475 6900 l 5521 6936 l 5568 6967 l 5615 6995 l 5660 7018 l
+ 5703 7036 l 5742 7051 l 5779 7062 l 5811 7069 l 5841 7073 l
+ 5867 7075 l 5891 7074 l 5912 7072 l 5931 7068 l 5948 7063 l
+ 5965 7057 l 5981 7050 l 5998 7043 l 6015 7036 l 6034 7030 l
+ 6054 7024 l 6077 7018 l 6104 7014 l 6133 7011 l 6167 7009 l
+ 6206 7009 l 6249 7010 l 6298 7013 l 6351 7018 l 6409 7024 l
+ 6470 7032 l 6534 7041 l 6600 7050 l 6669 7060 l 6736 7070 l
+ 6799 7078 l 6856 7087 l 6908 7094 l 6954 7100 l 6995 7106 l
+ 7030 7110 l 7060 7114 l 7086 7117 l 7109 7119 l 7128 7121 l
+ 7145 7123 l 7161 7124 l 7175 7125 l 7189 7126 l 7203 7127 l
+ 7219 7129 l 7235 7131 l 7254 7133 l 7275 7136 l 7299 7140 l
+ 7325 7144 l 7355 7150 l 7388 7156 l 7423 7163 l 7461 7172 l
+ 7500 7180 l 7538 7190 l 7575 7200 l 7601 7208 l 7625 7216 l
+ 7646 7223 l 7664 7231 l 7680 7238 l 7693 7244 l 7704 7251 l
+ 7713 7257 l 7720 7263 l 7725 7269 l 7729 7274 l 7731 7280 l
+ 7731 7285 l 7731 7290 l 7730 7295 l 7727 7299 l 7724 7304 l
+ 7720 7308 l 7716 7313 l 7711 7317 l 7705 7321 l 7700 7324 l
+ 7694 7328 l 7688 7331 l 7683 7334 l 7677 7337 l 7672 7340 l
+ 7668 7342 l 7664 7344 l 7660 7346 l 7657 7347 l 7655 7348 l
+
+ 7650 7350 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 7765 7271 m 7665 7344 l 7787 7327 l  col-1 s
+% Polyline
+2 slj
+ [60] 0 sd
+gs  clippath
+6163 6180 m 6315 6180 l 6315 6120 l 6163 6120 l 6163 6120 l 6283 6150 l 6163 6180 l cp
+eoclip
+n 7950 5400 m 7951 5400 l 7952 5399 l 7954 5399 l 7956 5399 l 7959 5398 l
+ 7963 5397 l 7967 5396 l 7971 5395 l 7977 5394 l 7983 5393 l
+ 7989 5391 l 7996 5390 l 8003 5388 l 8011 5387 l 8019 5385 l
+ 8027 5383 l 8035 5382 l 8042 5380 l 8050 5379 l 8058 5378 l
+ 8065 5377 l 8072 5376 l 8078 5375 l 8083 5374 l 8088 5374 l
+ 8092 5374 l 8095 5374 l 8097 5375 l 8098 5376 l 8098 5378 l
+ 8097 5379 l 8094 5382 l 8089 5385 l 8084 5388 l 8076 5392 l
+ 8066 5397 l 8055 5402 l 8041 5408 l 8025 5415 l 8006 5423 l
+ 7985 5432 l 7962 5441 l 7935 5452 l 7907 5463 l 7875 5475 l
+ 7832 5491 l 7786 5508 l 7738 5526 l 7690 5543 l 7643 5561 l
+ 7596 5577 l 7550 5593 l 7506 5608 l 7463 5623 l 7423 5637 l
+ 7384 5649 l 7347 5661 l 7312 5673 l 7279 5684 l 7246 5694 l
+ 7215 5703 l 7185 5713 l 7155 5722 l 7125 5731 l 7095 5740 l
+ 7065 5750 l 7035 5759 l 7004 5769 l 6971 5779 l 6938 5790 l
+ 6903 5802 l 6866 5814 l 6827 5828 l 6787 5842 l 6744 5857 l
+ 6700 5873 l 6654 5889 l 6607 5907 l 6560 5925 l 6512 5944 l
+ 6464 5963 l 6418 5982 l 6375 6000 l 6342 6014 l 6312 6028 l
+ 6285 6040 l 6261 6052 l 6239 6063 l 6221 6073 l 6205 6082 l
+ 6191 6090 l 6180 6098 l 6171 6105 l 6163 6111 l 6158 6116 l
+ 6154 6121 l 6152 6126 l 6152 6130 l 6152 6133 l 6154 6136 l
+ 6157 6139 l 6161 6141 l 6166 6143 l 6172 6145 l 6178 6146 l
+ 6185 6148 l 6193 6149 l 6200 6149 l 6209 6150 l 6217 6151 l
+ 6225 6151 l 6233 6151 l 6241 6151 l 6249 6151 l 6257 6151 l
+ 6264 6151 l 6270 6151 l 6276 6151 l 6281 6151 l 6286 6151 l
+ 6290 6150 l 6293 6150 l
+ 6300 6150 l gs col-1 s gr gr
+ [] 0 sd
+% arrowhead
+0 slj
+n 6163 6180 m 6283 6150 l 6163 6120 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+2563 4980 m 2715 4980 l 2715 4920 l 2563 4920 l 2563 4920 l 2683 4950 l 2563 4980 l cp
+eoclip
+n 5175 4200 m 5176 4200 l 5177 4200 l 5179 4199 l 5182 4199 l 5186 4199 l
+ 5190 4198 l 5196 4197 l 5203 4196 l 5210 4195 l 5219 4194 l
+ 5229 4193 l 5240 4191 l 5251 4190 l 5263 4188 l 5276 4187 l
+ 5289 4185 l 5302 4183 l 5316 4182 l 5329 4180 l 5342 4179 l
+ 5355 4178 l 5367 4177 l 5379 4176 l 5389 4175 l 5399 4174 l
+ 5408 4174 l 5415 4174 l 5421 4174 l 5426 4175 l 5428 4176 l
+ 5429 4178 l 5428 4179 l 5425 4182 l 5420 4185 l 5411 4188 l
+ 5401 4192 l 5387 4197 l 5370 4202 l 5350 4208 l 5326 4215 l
+ 5299 4223 l 5267 4232 l 5232 4241 l 5192 4252 l 5148 4263 l
+ 5100 4275 l 5048 4288 l 4994 4301 l 4937 4315 l 4879 4329 l
+ 4820 4343 l 4762 4356 l 4704 4370 l 4647 4383 l 4592 4395 l
+ 4538 4407 l 4485 4419 l 4435 4430 l 4386 4440 l 4339 4450 l
+ 4293 4460 l 4249 4469 l 4207 4478 l 4166 4486 l 4126 4494 l
+ 4087 4502 l 4049 4509 l 4011 4517 l 3974 4524 l 3937 4531 l
+ 3901 4538 l 3864 4546 l 3826 4553 l 3788 4561 l 3749 4569 l
+ 3709 4577 l 3668 4585 l 3626 4594 l 3582 4603 l 3536 4613 l
+ 3489 4624 l 3440 4634 l 3390 4646 l 3337 4658 l 3283 4671 l
+ 3228 4684 l 3171 4697 l 3113 4712 l 3055 4726 l 2996 4741 l
+ 2938 4756 l 2881 4771 l 2827 4786 l 2775 4800 l 2727 4814 l
+ 2683 4827 l 2643 4839 l 2608 4850 l 2576 4861 l 2549 4871 l
+ 2525 4880 l 2505 4888 l 2488 4895 l 2474 4902 l 2464 4908 l
+ 2455 4914 l 2450 4919 l 2447 4923 l 2446 4927 l 2447 4931 l
+ 2449 4934 l 2454 4937 l 2460 4939 l 2467 4942 l 2476 4943 l
+ 2486 4945 l 2496 4946 l 2508 4948 l 2520 4949 l 2533 4949 l
+ 2546 4950 l 2559 4950 l 2573 4951 l 2586 4951 l 2599 4951 l
+ 2612 4951 l 2624 4951 l 2635 4951 l 2646 4951 l 2656 4951 l
+ 2665 4951 l 2672 4951 l 2679 4951 l 2685 4950 l 2689 4950 l
+ 2693 4950 l
+ 2700 4950 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 2563 4980 m 2683 4950 l 2563 4920 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+7908 7262 m 7773 7329 l 7800 7383 l 7935 7316 l 7935 7316 l 7815 7343 l 7908 7262 l cp
+eoclip
+n 5475 4200 m 5475 4199 l 5475 4198 l 5475 4197 l 5475 4195 l 5475 4193 l
+ 5475 4189 l 5475 4185 l 5475 4180 l 5475 4175 l 5475 4168 l
+ 5475 4161 l 5475 4154 l 5475 4146 l 5475 4137 l 5475 4128 l
+ 5475 4119 l 5475 4110 l 5475 4100 l 5475 4091 l 5475 4081 l
+ 5475 4073 l 5475 4064 l 5475 4056 l 5475 4049 l 5475 4042 l
+ 5475 4036 l 5475 4031 l 5475 4027 l 5475 4025 l 5475 4023 l
+ 5475 4024 l 5475 4027 l 5475 4032 l 5475 4038 l 5475 4047 l
+ 5475 4058 l 5475 4071 l 5475 4086 l 5475 4104 l 5475 4125 l
+ 5475 4149 l 5475 4176 l 5475 4206 l 5475 4239 l 5475 4275 l
+ 5475 4322 l 5475 4371 l 5475 4423 l 5475 4474 l 5475 4525 l
+ 5474 4576 l 5474 4624 l 5473 4671 l 5473 4716 l 5472 4758 l
+ 5471 4798 l 5471 4836 l 5470 4871 l 5469 4905 l 5468 4937 l
+ 5467 4967 l 5466 4997 l 5465 5025 l 5464 5053 l 5462 5081 l
+ 5461 5109 l 5460 5138 l 5460 5167 l 5459 5197 l 5458 5229 l
+ 5457 5263 l 5457 5298 l 5457 5337 l 5456 5377 l 5457 5421 l
+ 5457 5468 l 5457 5518 l 5458 5571 l 5460 5627 l 5461 5685 l
+ 5463 5746 l 5466 5809 l 5468 5873 l 5471 5937 l 5475 6000 l
+ 5481 6085 l 5486 6164 l 5491 6234 l 5495 6294 l 5498 6346 l
+ 5499 6388 l 5499 6422 l 5497 6448 l 5494 6468 l 5490 6483 l
+ 5485 6493 l 5480 6499 l 5474 6504 l 5469 6506 l 5463 6508 l
+ 5459 6511 l 5456 6514 l 5454 6519 l 5454 6527 l 5457 6538 l
+ 5462 6552 l 5472 6571 l 5485 6593 l 5503 6620 l 5526 6650 l
+ 5554 6683 l 5587 6717 l 5625 6750 l 5664 6779 l 5705 6804 l
+ 5745 6826 l 5784 6844 l 5820 6858 l 5854 6870 l 5884 6878 l
+ 5911 6883 l 5934 6885 l 5955 6886 l 5973 6885 l 5989 6882 l
+ 6004 6878 l 6018 6874 l 6031 6869 l 6045 6864 l 6060 6859 l
+ 6076 6854 l 6095 6850 l 6117 6847 l 6142 6845 l 6172 6844 l
+ 6207 6845 l 6247 6848 l 6292 6852 l 6344 6858 l 6402 6867 l
+ 6464 6876 l 6531 6888 l 6600 6900 l 6666 6912 l 6730 6923 l
+ 6792 6935 l 6850 6945 l 6904 6955 l 6953 6963 l 6997 6971 l
+ 7037 6978 l 7072 6984 l 7104 6988 l 7132 6993 l 7157 6996 l
+ 7179 6999 l 7200 7002 l 7219 7004 l 7238 7006 l 7256 7008 l
+ 7274 7011 l 7293 7014 l 7314 7017 l 7336 7021 l 7360 7025 l
+ 7387 7030 l 7416 7037 l 7448 7044 l 7483 7052 l 7521 7062 l
+ 7561 7073 l 7602 7085 l 7644 7097 l 7686 7111 l 7725 7125 l
+ 7757 7137 l 7785 7150 l 7810 7162 l 7831 7173 l 7849 7184 l
+ 7863 7195 l 7875 7205 l 7885 7215 l 7892 7224 l 7896 7233 l
+ 7899 7241 l 7900 7250 l 7900 7258 l 7898 7265 l 7895 7273 l
+ 7891 7280 l 7886 7287 l 7880 7294 l 7873 7300 l 7866 7306 l
+ 7859 7312 l 7852 7318 l 7844 7323 l 7837 7328 l 7831 7332 l
+ 7824 7336 l 7819 7340 l 7814 7342 l 7810 7345 l 7806 7347 l
+
+ 7800 7350 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 7908 7262 m 7815 7343 l 7935 7316 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5383 8423 m 5292 8544 l 5340 8580 l 5431 8459 l 5431 8459 l 5335 8537 l 5383 8423 l cp
+eoclip
+n 4500 7800 m 4499 7800 l 4498 7799 l 4497 7799 l 4496 7799 l 4494 7798 l
+ 4493 7798 l 4491 7797 l 4489 7797 l 4487 7796 l 4484 7795 l
+ 4482 7795 l 4479 7794 l 4477 7793 l 4474 7793 l 4472 7792 l
+ 4469 7792 l 4467 7791 l 4465 7791 l 4462 7791 l 4461 7791 l
+ 4459 7791 l 4458 7791 l 4457 7791 l 4456 7792 l 4456 7793 l
+ 4456 7794 l 4457 7795 l 4458 7797 l 4460 7799 l 4462 7801 l
+ 4466 7804 l 4469 7807 l 4474 7810 l 4480 7814 l 4486 7819 l
+ 4493 7824 l 4502 7829 l 4511 7835 l 4521 7842 l 4533 7849 l
+ 4546 7857 l 4560 7866 l 4575 7875 l 4605 7893 l 4637 7912 l
+ 4670 7930 l 4702 7947 l 4732 7963 l 4760 7978 l 4787 7991 l
+ 4811 8002 l 4834 8012 l 4855 8021 l 4875 8029 l 4894 8037 l
+ 4913 8044 l 4931 8051 l 4950 8059 l 4970 8067 l 4991 8077 l
+ 5014 8088 l 5038 8101 l 5065 8116 l 5093 8133 l 5123 8153 l
+ 5155 8175 l 5188 8199 l 5220 8224 l 5250 8250 l 5279 8278 l
+ 5304 8304 l 5323 8329 l 5339 8351 l 5351 8372 l 5359 8391 l
+ 5365 8409 l 5368 8426 l 5369 8441 l 5368 8456 l 5366 8469 l
+ 5362 8482 l 5358 8494 l 5353 8505 l 5348 8515 l 5343 8524 l
+ 5338 8531 l 5334 8538 l 5331 8542 l
+ 5325 8550 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5383 8423 m 5335 8537 l 5431 8459 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5369 8459 m 5460 8580 l 5508 8544 l 5417 8423 l 5417 8423 l 5465 8537 l 5369 8459 l cp
+eoclip
+n 6300 7800 m 6301 7800 l 6302 7799 l 6303 7799 l 6304 7799 l 6306 7798 l
+ 6307 7798 l 6309 7797 l 6311 7797 l 6313 7796 l 6316 7795 l
+ 6318 7795 l 6321 7794 l 6323 7793 l 6326 7793 l 6328 7792 l
+ 6331 7792 l 6333 7791 l 6335 7791 l 6338 7791 l 6339 7791 l
+ 6341 7791 l 6342 7791 l 6343 7791 l 6344 7792 l 6344 7793 l
+ 6344 7794 l 6343 7795 l 6342 7797 l 6340 7799 l 6338 7801 l
+ 6334 7804 l 6331 7807 l 6326 7810 l 6320 7814 l 6314 7819 l
+ 6307 7824 l 6298 7829 l 6289 7835 l 6279 7842 l 6267 7849 l
+ 6254 7857 l 6240 7866 l 6225 7875 l 6195 7893 l 6163 7912 l
+ 6130 7930 l 6098 7947 l 6068 7963 l 6040 7978 l 6013 7991 l
+ 5989 8002 l 5966 8012 l 5945 8021 l 5925 8029 l 5906 8037 l
+ 5887 8044 l 5869 8051 l 5850 8059 l 5830 8067 l 5809 8077 l
+ 5786 8088 l 5762 8101 l 5735 8116 l 5707 8133 l 5677 8153 l
+ 5645 8175 l 5612 8199 l 5580 8224 l 5550 8250 l 5521 8278 l
+ 5496 8304 l 5477 8329 l 5461 8351 l 5449 8372 l 5441 8391 l
+ 5435 8409 l 5432 8426 l 5431 8441 l 5432 8456 l 5434 8469 l
+ 5438 8482 l 5442 8494 l 5447 8505 l 5452 8515 l 5457 8524 l
+ 5462 8531 l 5466 8538 l 5469 8542 l
+ 5475 8550 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5369 8459 m 5465 8537 l 5417 8423 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5288 8468 m 5149 8528 l 5173 8583 l 5312 8523 l 5312 8523 l 5190 8543 l 5288 8468 l cp
+eoclip
+n 2709 7780 m 2708 7780 l 2707 7780 l 2705 7780 l 2703 7780 l 2701 7781 l
+ 2697 7781 l 2694 7781 l 2690 7782 l 2685 7782 l 2680 7783 l
+ 2674 7783 l 2668 7784 l 2662 7785 l 2656 7786 l 2650 7787 l
+ 2644 7788 l 2638 7789 l 2632 7790 l 2626 7792 l 2621 7793 l
+ 2616 7795 l 2612 7796 l 2609 7798 l 2606 7800 l 2604 7803 l
+ 2604 7805 l 2604 7808 l 2605 7810 l 2607 7813 l 2611 7817 l
+ 2616 7820 l 2623 7824 l 2632 7828 l 2642 7833 l 2654 7838 l
+ 2669 7843 l 2685 7849 l 2704 7855 l 2725 7861 l 2749 7868 l
+ 2775 7875 l 2814 7885 l 2855 7895 l 2897 7906 l 2938 7915 l
+ 2977 7924 l 3014 7932 l 3047 7940 l 3078 7946 l 3106 7952 l
+ 3131 7957 l 3154 7961 l 3174 7965 l 3193 7968 l 3210 7971 l
+ 3227 7974 l 3243 7977 l 3260 7979 l 3277 7982 l 3296 7985 l
+ 3317 7989 l 3341 7993 l 3369 7998 l 3400 8004 l 3435 8010 l
+ 3476 8018 l 3522 8027 l 3573 8037 l 3629 8048 l 3691 8060 l
+ 3758 8072 l 3828 8086 l 3900 8100 l 3968 8113 l 4035 8126 l
+ 4099 8138 l 4160 8149 l 4216 8160 l 4268 8169 l 4315 8177 l
+ 4357 8184 l 4395 8190 l 4429 8195 l 4459 8199 l 4485 8202 l
+ 4509 8205 l 4531 8208 l 4551 8209 l 4570 8211 l 4588 8213 l
+ 4605 8214 l 4623 8216 l 4641 8217 l 4661 8220 l 4682 8223 l
+ 4705 8226 l 4730 8230 l 4758 8235 l 4788 8241 l 4822 8248 l
+ 4857 8256 l 4896 8265 l 4936 8276 l 4977 8287 l 5020 8299 l
+ 5061 8312 l 5100 8325 l 5134 8337 l 5164 8350 l 5190 8362 l
+ 5212 8373 l 5231 8384 l 5247 8395 l 5259 8405 l 5269 8415 l
+ 5276 8424 l 5281 8433 l 5284 8441 l 5285 8450 l 5284 8458 l
+ 5282 8465 l 5279 8473 l 5274 8480 l 5268 8487 l 5262 8494 l
+ 5255 8500 l 5247 8506 l 5239 8512 l 5231 8518 l 5223 8523 l
+ 5216 8528 l 5208 8532 l 5201 8536 l 5195 8540 l 5190 8542 l
+ 5186 8545 l 5182 8547 l
+ 5175 8550 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5288 8468 m 5190 8543 l 5312 8523 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5487 8523 m 5626 8583 l 5650 8528 l 5511 8468 l 5511 8468 l 5610 8543 l 5487 8523 l cp
+eoclip
+n 8100 7800 m 8101 7800 l 8102 7800 l 8103 7800 l 8105 7800 l 8107 7800 l
+ 8110 7800 l 8113 7800 l 8117 7800 l 8121 7800 l 8125 7800 l
+ 8130 7801 l 8136 7801 l 8141 7801 l 8147 7801 l 8152 7802 l
+ 8158 7802 l 8164 7803 l 8169 7803 l 8175 7804 l 8180 7805 l
+ 8184 7806 l 8189 7806 l 8192 7808 l 8195 7809 l 8198 7810 l
+ 8199 7812 l 8200 7813 l 8200 7815 l 8199 7817 l 8197 7819 l
+ 8194 7822 l 8189 7824 l 8183 7827 l 8176 7830 l 8167 7834 l
+ 8156 7838 l 8144 7842 l 8129 7847 l 8112 7852 l 8094 7857 l
+ 8073 7863 l 8050 7869 l 8025 7875 l 7986 7885 l 7944 7894 l
+ 7903 7904 l 7862 7913 l 7822 7922 l 7785 7930 l 7751 7937 l
+ 7721 7944 l 7693 7949 l 7668 7954 l 7645 7959 l 7625 7963 l
+ 7606 7966 l 7589 7969 l 7572 7972 l 7556 7975 l 7540 7978 l
+ 7522 7981 l 7503 7984 l 7482 7988 l 7458 7993 l 7431 7998 l
+ 7400 8003 l 7365 8010 l 7324 8018 l 7278 8027 l 7227 8036 l
+ 7171 8047 l 7109 8059 l 7042 8072 l 6972 8086 l 6900 8100 l
+ 6832 8113 l 6765 8126 l 6701 8138 l 6640 8149 l 6584 8160 l
+ 6532 8169 l 6485 8177 l 6443 8184 l 6405 8190 l 6371 8195 l
+ 6341 8199 l 6315 8202 l 6291 8205 l 6269 8208 l 6249 8209 l
+ 6230 8211 l 6212 8213 l 6195 8214 l 6177 8216 l 6159 8217 l
+ 6139 8220 l 6118 8223 l 6095 8226 l 6070 8230 l 6042 8235 l
+ 6012 8241 l 5978 8248 l 5943 8256 l 5904 8265 l 5864 8276 l
+ 5823 8287 l 5780 8299 l 5739 8312 l 5700 8325 l 5666 8337 l
+ 5636 8350 l 5610 8362 l 5588 8373 l 5569 8384 l 5553 8395 l
+ 5541 8405 l 5531 8415 l 5524 8424 l 5519 8433 l 5516 8441 l
+ 5515 8450 l 5516 8458 l 5518 8465 l 5521 8473 l 5526 8480 l
+ 5532 8487 l 5538 8494 l 5545 8500 l 5553 8506 l 5561 8512 l
+ 5569 8518 l 5577 8523 l 5584 8528 l 5592 8532 l 5599 8536 l
+ 5605 8540 l 5610 8542 l 5614 8545 l 5618 8547 l
+ 5625 8550 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5487 8523 m 5610 8543 l 5511 8468 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5639 8516 m 5775 8583 l 5801 8529 l 5666 8462 l 5666 8462 l 5760 8543 l 5639 8516 l cp
+eoclip
+n 9900 7800 m 9901 7800 l 9902 7800 l 9903 7800 l 9905 7800 l 9907 7800 l
+ 9910 7800 l 9913 7800 l 9917 7800 l 9921 7800 l 9925 7800 l
+ 9930 7801 l 9936 7801 l 9941 7801 l 9947 7801 l 9952 7802 l
+ 9958 7802 l 9964 7803 l 9969 7803 l 9975 7804 l 9980 7805 l
+ 9984 7806 l 9989 7806 l 9992 7808 l 9995 7809 l 9998 7810 l
+ 9999 7812 l 10000 7813 l 10000 7815 l 9999 7817 l 9997 7819 l
+ 9994 7822 l 9989 7824 l 9983 7827 l 9976 7830 l 9967 7834 l
+ 9956 7838 l 9944 7842 l 9929 7847 l 9912 7852 l 9894 7857 l
+ 9873 7863 l 9850 7869 l 9825 7875 l 9786 7885 l 9745 7894 l
+ 9703 7904 l 9663 7913 l 9626 7922 l 9591 7931 l 9560 7938 l
+ 9532 7945 l 9508 7951 l 9487 7957 l 9469 7962 l 9454 7966 l
+ 9440 7970 l 9428 7974 l 9417 7978 l 9406 7981 l 9395 7985 l
+ 9382 7988 l 9368 7992 l 9350 7997 l 9329 8001 l 9304 8007 l
+ 9274 8013 l 9238 8019 l 9196 8027 l 9147 8035 l 9090 8044 l
+ 9026 8054 l 8954 8065 l 8875 8076 l 8790 8088 l 8700 8100 l
+ 8630 8109 l 8560 8117 l 8490 8125 l 8423 8133 l 8359 8140 l
+ 8298 8146 l 8241 8152 l 8188 8157 l 8138 8162 l 8093 8166 l
+ 8052 8169 l 8015 8172 l 7981 8174 l 7951 8176 l 7923 8177 l
+ 7897 8179 l 7874 8179 l 7853 8180 l 7832 8181 l 7813 8181 l
+ 7794 8181 l 7775 8182 l 7755 8182 l 7735 8182 l 7714 8183 l
+ 7690 8184 l 7665 8185 l 7638 8186 l 7607 8188 l 7574 8190 l
+ 7537 8192 l 7496 8195 l 7452 8198 l 7403 8202 l 7351 8207 l
+ 7294 8211 l 7234 8217 l 7171 8223 l 7105 8229 l 7037 8236 l
+ 6968 8243 l 6900 8250 l 6813 8259 l 6730 8268 l 6654 8277 l
+ 6585 8285 l 6523 8292 l 6469 8298 l 6423 8303 l 6383 8308 l
+ 6349 8312 l 6321 8315 l 6298 8318 l 6280 8320 l 6264 8321 l
+ 6252 8323 l 6241 8324 l 6231 8325 l 6222 8326 l 6213 8327 l
+ 6203 8329 l 6191 8330 l 6178 8332 l 6162 8335 l 6144 8338 l
+ 6122 8342 l 6097 8347 l 6068 8352 l 6036 8358 l 6001 8365 l
+ 5964 8373 l 5925 8382 l 5887 8391 l 5850 8400 l 5823 8407 l
+ 5799 8415 l 5777 8422 l 5758 8429 l 5742 8436 l 5728 8442 l
+ 5716 8449 l 5706 8455 l 5699 8460 l 5693 8466 l 5689 8471 l
+ 5686 8476 l 5684 8481 l 5684 8486 l 5685 8491 l 5687 8496 l
+ 5690 8500 l 5693 8504 l 5698 8508 l 5702 8513 l 5708 8516 l
+ 5713 8520 l 5719 8524 l 5725 8527 l 5731 8530 l 5737 8533 l
+ 5743 8536 l 5748 8539 l 5753 8541 l 5758 8543 l 5762 8545 l
+ 5765 8546 l 5768 8547 l 5771 8548 l
+ 5775 8550 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5639 8516 m 5760 8543 l 5666 8462 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5133 8462 m 4998 8529 l 5025 8583 l 5160 8516 l 5160 8516 l 5040 8543 l 5133 8462 l cp
+eoclip
+n 900 7800 m 899 7800 l 898 7800 l 897 7800 l 895 7800 l 893 7800 l
+ 890 7800 l 887 7800 l 883 7800 l 879 7800 l 875 7800 l
+ 870 7801 l 864 7801 l 859 7801 l 853 7801 l 848 7802 l
+ 842 7802 l 836 7803 l 831 7803 l 825 7804 l 820 7805 l
+ 816 7806 l 811 7806 l 808 7808 l 805 7809 l 802 7810 l
+ 801 7812 l 800 7813 l 800 7815 l 801 7817 l 803 7819 l
+ 806 7822 l 811 7824 l 817 7827 l 824 7830 l 833 7834 l
+ 844 7838 l 856 7842 l 871 7847 l 888 7852 l 906 7857 l
+ 927 7863 l 950 7869 l 975 7875 l 1014 7885 l 1055 7894 l
+ 1097 7904 l 1137 7913 l 1174 7922 l 1209 7931 l 1240 7938 l
+ 1268 7945 l 1292 7951 l 1313 7957 l 1331 7962 l 1346 7966 l
+ 1360 7970 l 1372 7974 l 1383 7978 l 1394 7981 l 1405 7985 l
+ 1418 7988 l 1432 7992 l 1450 7997 l 1471 8001 l 1496 8007 l
+ 1526 8013 l 1562 8019 l 1604 8027 l 1653 8035 l 1710 8044 l
+ 1774 8054 l 1846 8065 l 1925 8076 l 2010 8088 l 2100 8100 l
+ 2170 8109 l 2240 8117 l 2310 8125 l 2377 8133 l 2441 8140 l
+ 2502 8146 l 2559 8152 l 2612 8157 l 2662 8162 l 2707 8166 l
+ 2748 8169 l 2785 8172 l 2819 8174 l 2849 8176 l 2877 8177 l
+ 2903 8179 l 2926 8179 l 2947 8180 l 2968 8181 l 2987 8181 l
+ 3006 8181 l 3025 8182 l 3045 8182 l 3065 8182 l 3086 8183 l
+ 3110 8184 l 3135 8185 l 3162 8186 l 3193 8188 l 3226 8190 l
+ 3263 8192 l 3304 8195 l 3348 8198 l 3397 8202 l 3449 8207 l
+ 3506 8211 l 3566 8217 l 3629 8223 l 3695 8229 l 3763 8236 l
+ 3832 8243 l 3900 8250 l 3987 8259 l 4070 8268 l 4146 8277 l
+ 4215 8285 l 4277 8292 l 4331 8298 l 4377 8303 l 4417 8308 l
+ 4451 8312 l 4479 8315 l 4502 8318 l 4520 8320 l 4536 8321 l
+ 4548 8323 l 4559 8324 l 4569 8325 l 4578 8326 l 4587 8327 l
+ 4597 8329 l 4609 8330 l 4622 8332 l 4638 8335 l 4656 8338 l
+ 4678 8342 l 4703 8347 l 4732 8352 l 4764 8358 l 4799 8365 l
+ 4836 8373 l 4875 8382 l 4913 8391 l 4950 8400 l 4977 8407 l
+ 5001 8415 l 5023 8422 l 5042 8429 l 5058 8436 l 5072 8442 l
+ 5084 8449 l 5094 8455 l 5101 8460 l 5107 8466 l 5111 8471 l
+ 5114 8476 l 5116 8481 l 5116 8486 l 5115 8491 l 5113 8496 l
+ 5110 8500 l 5107 8504 l 5102 8508 l 5098 8513 l 5092 8516 l
+ 5087 8520 l 5081 8524 l 5075 8527 l 5069 8530 l 5063 8533 l
+ 5057 8536 l 5052 8539 l 5047 8541 l 5042 8543 l 5038 8545 l
+ 5035 8546 l 5032 8547 l 5029 8548 l
+ 5025 8550 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5133 8462 m 5040 8543 l 5160 8516 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+5537 3720 m 5385 3720 l 5385 3780 l 5537 3780 l 5537 3780 l 5417 3750 l 5537 3720 l cp
+eoclip
+n 3225 3000 m 3224 3000 l 3223 3000 l 3222 2999 l 3219 2999 l 3216 2999 l
+ 3212 2998 l 3207 2997 l 3201 2996 l 3195 2995 l 3187 2994 l
+ 3179 2993 l 3170 2991 l 3160 2990 l 3149 2988 l 3139 2987 l
+ 3127 2985 l 3116 2983 l 3105 2982 l 3093 2980 l 3082 2979 l
+ 3071 2978 l 3061 2977 l 3051 2976 l 3042 2975 l 3034 2974 l
+ 3027 2974 l 3021 2974 l 3016 2974 l 3012 2975 l 3010 2976 l
+ 3009 2978 l 3011 2979 l 3014 2982 l 3019 2985 l 3026 2988 l
+ 3036 2992 l 3048 2997 l 3063 3002 l 3081 3008 l 3102 3015 l
+ 3126 3023 l 3153 3032 l 3185 3041 l 3220 3052 l 3258 3063 l
+ 3300 3075 l 3350 3089 l 3402 3104 l 3456 3119 l 3512 3134 l
+ 3568 3149 l 3623 3164 l 3677 3178 l 3731 3192 l 3782 3205 l
+ 3832 3218 l 3881 3230 l 3927 3241 l 3971 3252 l 4014 3263 l
+ 4055 3272 l 4095 3282 l 4133 3291 l 4171 3299 l 4207 3307 l
+ 4242 3315 l 4278 3323 l 4313 3331 l 4347 3339 l 4383 3347 l
+ 4418 3355 l 4454 3364 l 4492 3372 l 4530 3381 l 4570 3391 l
+ 4611 3401 l 4654 3411 l 4698 3423 l 4744 3434 l 4793 3447 l
+ 4843 3460 l 4894 3474 l 4948 3489 l 5002 3504 l 5057 3519 l
+ 5113 3536 l 5169 3552 l 5223 3568 l 5275 3584 l 5325 3600 l
+ 5367 3614 l 5405 3627 l 5440 3639 l 5472 3650 l 5499 3661 l
+ 5523 3671 l 5544 3680 l 5562 3688 l 5577 3695 l 5589 3702 l
+ 5599 3708 l 5606 3714 l 5611 3719 l 5614 3723 l 5616 3727 l
+ 5615 3731 l 5613 3734 l 5609 3737 l 5604 3739 l 5598 3742 l
+ 5591 3743 l 5583 3745 l 5574 3746 l 5564 3748 l 5554 3749 l
+ 5543 3749 l 5532 3750 l 5520 3750 l 5509 3751 l 5498 3751 l
+ 5486 3751 l 5476 3751 l 5465 3751 l 5455 3751 l 5446 3751 l
+ 5438 3751 l 5430 3751 l 5424 3751 l 5418 3751 l 5413 3750 l
+ 5409 3750 l 5406 3750 l
+ 5400 3750 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 5537 3720 m 5417 3750 l 5537 3780 l  col-1 s
+% Polyline
+2 slj
+gs  clippath
+877 7487 m 933 7347 l 877 7324 l 821 7465 l 821 7465 l 894 7365 l 877 7487 l cp
+eoclip
+n 3075 3000 m 3075 2999 l 3076 2999 l 3077 2998 l 3078 2996 l 3080 2994 l
+ 3082 2991 l 3085 2988 l 3088 2984 l 3091 2980 l 3095 2975 l
+ 3100 2969 l 3105 2963 l 3110 2957 l 3116 2950 l 3121 2943 l
+ 3127 2935 l 3133 2928 l 3139 2921 l 3145 2913 l 3151 2906 l
+ 3156 2899 l 3162 2892 l 3167 2886 l 3171 2880 l 3175 2875 l
+ 3179 2871 l 3181 2867 l 3183 2864 l 3185 2862 l 3185 2861 l
+ 3184 2862 l 3183 2863 l 3180 2866 l 3176 2870 l 3171 2876 l
+ 3164 2883 l 3156 2892 l 3147 2904 l 3135 2917 l 3122 2932 l
+ 3107 2950 l 3090 2970 l 3070 2992 l 3049 3017 l 3026 3045 l
+ 3000 3075 l 2969 3112 l 2935 3151 l 2901 3190 l 2867 3229 l
+ 2833 3268 l 2800 3304 l 2768 3338 l 2737 3370 l 2708 3399 l
+ 2680 3425 l 2654 3448 l 2630 3469 l 2607 3487 l 2586 3503 l
+ 2566 3517 l 2547 3529 l 2529 3540 l 2512 3549 l 2495 3558 l
+ 2479 3567 l 2462 3575 l 2446 3584 l 2429 3593 l 2412 3604 l
+ 2394 3617 l 2375 3632 l 2354 3650 l 2332 3671 l 2308 3695 l
+ 2282 3723 l 2254 3757 l 2224 3795 l 2191 3838 l 2156 3888 l
+ 2119 3943 l 2078 4005 l 2036 4073 l 1991 4148 l 1945 4228 l
+ 1897 4314 l 1848 4405 l 1800 4500 l 1761 4579 l 1724 4658 l
+ 1687 4737 l 1652 4816 l 1619 4893 l 1587 4968 l 1558 5041 l
+ 1530 5110 l 1504 5177 l 1480 5241 l 1459 5302 l 1439 5359 l
+ 1420 5414 l 1404 5465 l 1389 5513 l 1375 5559 l 1363 5602 l
+ 1352 5643 l 1342 5682 l 1333 5718 l 1325 5753 l 1318 5787 l
+ 1311 5819 l 1305 5851 l 1299 5882 l 1294 5913 l 1288 5943 l
+ 1283 5973 l 1277 6004 l 1271 6036 l 1265 6068 l 1259 6102 l
+ 1252 6136 l 1244 6173 l 1236 6211 l 1227 6250 l 1217 6292 l
+ 1206 6336 l 1195 6382 l 1182 6430 l 1168 6481 l 1154 6534 l
+ 1138 6589 l 1122 6646 l 1105 6705 l 1087 6765 l 1068 6826 l
+ 1049 6888 l 1030 6949 l 1011 7010 l 993 7069 l 975 7125 l
+ 958 7178 l 943 7227 l 928 7273 l 915 7314 l 903 7352 l
+ 893 7385 l 883 7416 l 875 7442 l 867 7466 l 861 7486 l
+ 855 7503 l 850 7518 l 847 7530 l 843 7539 l 841 7546 l
+ 839 7552 l 838 7555 l 838 7556 l 837 7556 l 838 7554 l
+ 839 7551 l 840 7547 l 842 7541 l 844 7535 l 846 7527 l
+ 848 7519 l 851 7510 l 854 7500 l 857 7490 l 860 7480 l
+ 864 7469 l 867 7458 l 870 7447 l 873 7437 l 876 7427 l
+ 879 7417 l 882 7407 l 885 7399 l 888 7391 l 890 7383 l
+ 892 7376 l 894 7371 l 895 7365 l 897 7361 l 898 7358 l
+ 898 7355 l
+ 900 7350 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 877 7487 m 894 7365 l 821 7465 l  col-1 s
+/Times-Roman ff 180.00 scf sf
+9480 6442 m
+gs 1 -1 sc (LChannelR) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+7680 6442 m
+gs 1 -1 sc (LChannelC) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+7740 5242 m
+gs 1 -1 sc (LChannel) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+5820 6442 m
+gs 1 -1 sc (LChannelCR) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+7965 7642 m
+gs 1 -1 sc (Blit) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+4072 5242 m
+gs 1 -1 sc (NTChannel) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+4027 6420 m
+gs 1 -1 sc (Lifting inline) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+4282 7642 m
+gs 1 -1 sc (Timer) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+6007 7620 m
+gs 1 -1 sc (Storage) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+9697 7642 m
+gs 1 -1 sc (Color) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+2452 7620 m
+gs 1 -1 sc (Lifting) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+2377 5242 m
+gs 1 -1 sc (Wavelet) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+5092 4042 m
+gs 1 -1 sc (Channel) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+757 7642 m
+gs 1 -1 sc (Util) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+5167 8820 m
+gs 1 -1 sc (Types) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+5062 10020 m
+gs 1 -1 sc (Compiler) col-1 sh gr
+/Times-Roman ff 180.00 scf sf
+2917 2820 m
+gs 1 -1 sc (Image) col-1 sh gr
+% here ends figure;
+pagefooter
+showpage
+%%Trailer
+%EOF
--- waili-19990723.orig/man/Includes.fig
+++ waili-19990723/man/Includes.fig
@@ -1,4 +1,11 @@
-#FIG 3.1
+#FIG 3.2
+Portrait
+Center
+Inches
+A4      
+100.00
+Single
+-2
 # 
 #   Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium
 # 
@@ -16,9 +23,6 @@
 #   along with this program; if not, write to the Free Software
 #   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 # 
-Portrait
-Center
-Inches
 1200 2
 6 9300 6150 10500 6600
 2 2 0 1 -1 7 0 0 -1 0.000 0 0 -1 0 0 5
@@ -147,102 +151,74 @@ Inches
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 6
 	0 0 1.00 60.00 120.00
 	 8250 5400 8325 5550 8850 6000 8850 6675 8325 7050 8250 7350
-	 2100.00 0.00 8287.77 5488.84 8306.52 5526.34 8426.57 5680.09
-	 8769.64 5825.28 8928.45 6170.56 8939.19 6501.68 8764.75 6840.66
-	 8420.01 6910.44 8290.84 7100.17 8272.09 7175.17 2100.00 0.00
+	 0.000 -1.000 -1.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 6300 6600 6375 6675 7875 7050 7950 7350
-	 2100.00 0.00 6339.10 6646.54 6357.85 6665.29 6708.37 6863.75
-	 7540.24 6715.25 7925.21 7100.21 7943.96 7175.21 2100.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 9900 6600 9825 6675 8475 7125 8400 7350
-	 2100.00 0.00 9860.16 6646.11 9841.41 6664.86 9531.43 6856.44
-	 8762.65 6837.34 8439.04 7160.96 8420.29 7217.21 2100.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 1 1 -1 7 0 0 -1 4.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 8400 5400 8475 5475 9825 6000 9900 6150
-	 2100.00 0.00 8440.29 5445.84 8459.04 5464.59 8765.63 5664.64
-	 9541.65 5741.79 9849.60 6022.42 9868.35 6059.92 2100.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 1 1 -1 7 0 0 -1 4.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 5625 4200 5700 4275 8025 4800 8100 4950
-	 0.00 0.00 5663.87 4246.67 5682.62 4265.42 6220.79 4562.04
-	 7498.17 4387.23 8052.80 4821.78 8071.55 4859.28 0.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 1 1 -1 7 0 0 -1 4.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 5325 4200 5250 4350 4575 4800 4500 4950
-	 0.00 0.00 5289.25 4289.28 5270.50 4326.78 5117.80 4499.75
-	 4707.20 4650.24 4554.50 4823.22 4535.75 4860.72 0.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 7
 	0 0 1.00 60.00 120.00
 	 4650 5400 4725 5550 5175 5850 5475 6900 6600 7050 7575 7200
 	 7650 7350
-	 0.00 0.00 4685.75 5489.28 4704.50 5526.78 4813.13 5649.84
-	 5091.59 5735.73 5343.42 6080.72 5212.43 6673.16 5747.90 7135.76
-	 6344.04 7013.25 6822.47 7081.94 7344.41 7031.91 7604.40 7221.43
-	 7623.15 7258.93 0.00 0.00
+	 0.000 -1.000 -1.000 -1.000 -1.000 -1.000 0.000
 3 2 1 1 -1 7 0 0 -1 4.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 7950 5400 7875 5475 6375 6000 6300 6150
-	 2100.00 0.00 7910.02 5446.02 7891.27 5464.77 7549.91 5679.29
-	 6694.80 5718.42 6349.69 6022.29 6330.94 6059.79 2100.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 5175 4200 5100 4275 2775 4800 2700 4950
-	 0.00 0.00 5136.13 4246.67 5117.38 4265.42 4579.22 4562.04
-	 3301.82 4387.21 2747.20 4821.78 2728.45 4859.28 0.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 7
 	0 0 1.00 60.00 120.00
 	 5475 4200 5475 4275 5475 6000 5625 6750 6600 6900 7725 7125
 	 7800 7350
-	 0.00 0.00 5475.00 4243.44 5475.00 4262.19 5475.00 4667.97
-	 5435.82 5604.35 5492.37 6175.43 5444.17 6577.09 5858.23 6973.02
-	 6378.57 6860.85 6857.52 6945.53 7465.16 6895.62 7765.29 7160.56
-	 7784.04 7216.81 0.00 0.00
+	 0.000 -1.000 -1.000 -1.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 4500 7800 4575 7875 5250 8250 5325 8550
-	 -1500.00 0.00 4541.44 7845.08 4560.19 7863.83 4718.71 7983.40
-	 5121.58 8082.58 5288.57 8300.28 5307.32 8375.28 -1500.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 6300 7800 6225 7875 5550 8250 5475 8550
-	 -1500.00 0.00 6258.55 7845.08 6239.80 7863.83 6081.29 7983.40
-	 5678.42 8082.58 5511.43 8300.29 5492.68 8375.29 -1500.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 5
 	0 0 1.00 60.00 120.00
 	 2709 7780 2775 7875 3900 8100 5100 8325 5175 8550
-	 -1500.00 0.00 2739.08 7838.51 2755.58 7862.26 3031.80 8043.43
-	 3643.40 8050.29 4173.07 8152.90 4820.47 8081.23 5140.72 8360.52
-	 5159.47 8416.77 -1500.00 0.00
+	 0.000 -1.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 5
 	0 0 1.00 60.00 120.00
 	 8100 7800 8025 7875 6900 8100 5700 8325 5625 8550
-	 -1500.00 0.00 8061.38 7846.81 8042.63 7865.56 7770.69 8011.10
-	 7156.60 8050.28 6626.93 8152.91 5979.52 8081.22 5659.28 8360.52
-	 5640.53 8416.77 -1500.00 0.00
+	 0.000 -1.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 6
 	0 0 1.00 60.00 120.00
 	 9900 7800 9825 7875 8700 8100 6900 8250 5850 8400 5775 8550
-	 -1500.00 0.00 9861.38 7846.81 9842.63 7865.56 9570.69 8011.10
-	 8959.81 8063.31 8290.96 8157.76 7309.30 8203.74 6659.65 8277.16
-	 6100.07 8219.75 5820.34 8421.38 5801.59 8458.88 -1500.00 0.00
+	 0.000 -1.000 -1.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 6
 	0 0 1.00 60.00 120.00
 	 900 7800 975 7875 2100 8100 3900 8250 4950 8400 5025 8550
-	 -1500.00 0.00 938.62 7846.81 957.37 7865.56 1229.31 8011.10
-	 1840.19 8063.32 2509.04 8157.75 3490.70 8203.76 4140.35 8277.16
-	 4699.92 8219.76 4979.66 8421.37 4998.41 8458.87 -1500.00 0.00
+	 0.000 -1.000 -1.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 4
 	0 0 1.00 60.00 120.00
 	 3225 3000 3300 3075 5325 3600 5400 3750
-	 0.00 0.00 3264.18 3046.49 3282.93 3065.24 3748.78 3331.67
-	 4874.35 3235.28 5352.10 3621.93 5370.85 3659.43 0.00 0.00
+	 0.000 -1.000 -1.000 0.000
 3 2 0 1 -1 7 0 0 -1 0.000 0 1 0 5
 	0 0 1.00 60.00 120.00
 	 3075 3000 3000 3075 1800 4500 975 7125 900 7350
-	 0.00 0.00 3031.03 3042.87 3012.28 3061.62 2712.37 3388.34
-	 2013.89 4110.51 1484.09 5075.28 1168.10 6528.58 962.52 7163.56
-	 943.77 7219.81 0.00 0.00
+	 0.000 -1.000 -1.000 -1.000 0.000
--- /dev/null
+++ waili-19990723/man/Lifting_dep.eps
@@ -0,0 +1,157 @@
+%!PS-Adobe-2.0
+%%Title: Lifting_dep.fig
+%%Creator: fig2dev Version 3.2 Patchlevel 3b
+%%CreationDate: Thu May 17 12:30:00 2001
+%%For: debian@etabeta (A C G Mennucc,,,)
+%%Orientation: Portrait
+%%Pages: 1
+%%BoundingBox: 0 0 612 792
+%%BeginSetup
+%%IncludeFeature: *PageSize Letter
+%%EndSetup
+%%Magnification: 1.0000
+%%EndComments
+%
+%   
+%    Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium 
+%   
+%    This program is free software; you can redistribute it and/or modify 
+%    it under the terms of the GNU General Public License as published by 
+%    the Free Software Foundation; either version 2 of the License, or 
+%    (at your option) any later version. 
+%   
+%    This program is distributed in the hope that it will be useful, 
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of 
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+%    GNU General Public License for more details. 
+%   
+%    You should have received a copy of the GNU General Public License 
+%    along with this program; if not, write to the Free Software 
+%    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+%   
+%
+/$F2psDict 200 dict def
+$F2psDict begin
+$F2psDict /mtrx matrix put
+/col-1 {0 setgray} bind def
+/col0 {0.000 0.000 0.000 srgb} bind def
+/col1 {0.000 0.000 1.000 srgb} bind def
+/col2 {0.000 1.000 0.000 srgb} bind def
+/col3 {0.000 1.000 1.000 srgb} bind def
+/col4 {1.000 0.000 0.000 srgb} bind def
+/col5 {1.000 0.000 1.000 srgb} bind def
+/col6 {1.000 1.000 0.000 srgb} bind def
+/col7 {1.000 1.000 1.000 srgb} bind def
+/col8 {0.000 0.000 0.560 srgb} bind def
+/col9 {0.000 0.000 0.690 srgb} bind def
+/col10 {0.000 0.000 0.820 srgb} bind def
+/col11 {0.530 0.810 1.000 srgb} bind def
+/col12 {0.000 0.560 0.000 srgb} bind def
+/col13 {0.000 0.690 0.000 srgb} bind def
+/col14 {0.000 0.820 0.000 srgb} bind def
+/col15 {0.000 0.560 0.560 srgb} bind def
+/col16 {0.000 0.690 0.690 srgb} bind def
+/col17 {0.000 0.820 0.820 srgb} bind def
+/col18 {0.560 0.000 0.000 srgb} bind def
+/col19 {0.690 0.000 0.000 srgb} bind def
+/col20 {0.820 0.000 0.000 srgb} bind def
+/col21 {0.560 0.000 0.560 srgb} bind def
+/col22 {0.690 0.000 0.690 srgb} bind def
+/col23 {0.820 0.000 0.820 srgb} bind def
+/col24 {0.500 0.190 0.000 srgb} bind def
+/col25 {0.630 0.250 0.000 srgb} bind def
+/col26 {0.750 0.380 0.000 srgb} bind def
+/col27 {1.000 0.500 0.500 srgb} bind def
+/col28 {1.000 0.630 0.630 srgb} bind def
+/col29 {1.000 0.750 0.750 srgb} bind def
+/col30 {1.000 0.880 0.880 srgb} bind def
+/col31 {1.000 0.840 0.000 srgb} bind def
+
+end
+save
+newpath 0 792 moveto 0 0 lineto 612 0 lineto 612 792 lineto closepath clip newpath
+189.0 472.5 translate
+1 -1 scale
+
+/cp {closepath} bind def
+/ef {eofill} bind def
+/gr {grestore} bind def
+/gs {gsave} bind def
+/sa {save} bind def
+/rs {restore} bind def
+/l {lineto} bind def
+/m {moveto} bind def
+/rm {rmoveto} bind def
+/n {newpath} bind def
+/s {stroke} bind def
+/sh {show} bind def
+/slc {setlinecap} bind def
+/slj {setlinejoin} bind def
+/slw {setlinewidth} bind def
+/srgb {setrgbcolor} bind def
+/rot {rotate} bind def
+/sc {scale} bind def
+/sd {setdash} bind def
+/ff {findfont} bind def
+/sf {setfont} bind def
+/scf {scalefont} bind def
+/sw {stringwidth} bind def
+/tr {translate} bind def
+/tnt {dup dup currentrgbcolor
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add srgb}
+  bind def
+/shd {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
+  4 -2 roll mul srgb} bind def
+/$F2psBegin {$F2psDict begin /$F2psEnteredState save def} def
+/$F2psEnd {$F2psEnteredState restore end} def
+
+$F2psBegin
+%%Page: 1 1
+10 setmiterlimit
+ 0.06000 0.06000 sc
+%%Page: 1 1
+%
+% Fig objects follow
+%
+% Polyline
+7.500 slw
+n 450 1650 m 1725 1650 l 1725 2100 l 450 2100 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+600 1950 m
+gs 1 -1 sc (LiftChannelR) col-1 sh gr
+% Polyline
+n 2175 1650 m 3450 1650 l 3450 2100 l 2175 2100 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+2325 1950 m
+gs 1 -1 sc (LiftChannelC) col-1 sh gr
+% Polyline
+n 1425 450 m 2475 450 l 2475 900 l 1425 900 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+1725 750 m
+gs 1 -1 sc (Lifting) col-1 sh gr
+% Polyline
+gs  clippath
+1093 1637 m 1134 1682 l 1245 1580 l 1137 1639 l 1205 1536 l cp
+eoclip
+n 1950 900 m
+ 1125 1650 l gs col-1 s gr gr
+
+% arrowhead
+n 1205 1536 m 1137 1639 l 1245 1580 l 1205 1536 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+2765 1682 m 2806 1637 l 2694 1536 l 2763 1639 l 2654 1580 l cp
+eoclip
+n 1950 900 m
+ 2775 1650 l gs col-1 s gr gr
+
+% arrowhead
+n 2654 1580 m 2763 1639 l 2694 1536 l 2654 1580 l  cp gs 0.00 setgray ef gr  col-1 s
+$F2psEnd
+rs
+showpage
--- /dev/null
+++ waili-19990723/man/WAILI.eps
@@ -0,0 +1,173 @@
+%!PS-Adobe-2.0
+%%Title: WAILI.fig
+%%Creator: fig2dev Version 3.2 Patchlevel 3b
+%%CreationDate: Thu May 17 12:30:00 2001
+%%For: debian@etabeta (A C G Mennucc,,,)
+%%Orientation: Landscape
+%%Pages: 1
+%%BoundingBox: 0 0 595 842
+%%BeginSetup
+%%IncludeFeature: *PageSize A4
+%%EndSetup
+%%Magnification: 1.0000
+%%EndComments
+%
+%   
+%    Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium 
+%   
+%    This program is free software; you can redistribute it and/or modify 
+%    it under the terms of the GNU General Public License as published by 
+%    the Free Software Foundation; either version 2 of the License, or 
+%    (at your option) any later version. 
+%   
+%    This program is distributed in the hope that it will be useful, 
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of 
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+%    GNU General Public License for more details. 
+%   
+%    You should have received a copy of the GNU General Public License 
+%    along with this program; if not, write to the Free Software 
+%    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+%   
+%
+/$F2psDict 200 dict def
+$F2psDict begin
+$F2psDict /mtrx matrix put
+/col-1 {0 setgray} bind def
+/col0 {0.000 0.000 0.000 srgb} bind def
+/col1 {0.000 0.000 1.000 srgb} bind def
+/col2 {0.000 1.000 0.000 srgb} bind def
+/col3 {0.000 1.000 1.000 srgb} bind def
+/col4 {1.000 0.000 0.000 srgb} bind def
+/col5 {1.000 0.000 1.000 srgb} bind def
+/col6 {1.000 1.000 0.000 srgb} bind def
+/col7 {1.000 1.000 1.000 srgb} bind def
+/col8 {0.000 0.000 0.560 srgb} bind def
+/col9 {0.000 0.000 0.690 srgb} bind def
+/col10 {0.000 0.000 0.820 srgb} bind def
+/col11 {0.530 0.810 1.000 srgb} bind def
+/col12 {0.000 0.560 0.000 srgb} bind def
+/col13 {0.000 0.690 0.000 srgb} bind def
+/col14 {0.000 0.820 0.000 srgb} bind def
+/col15 {0.000 0.560 0.560 srgb} bind def
+/col16 {0.000 0.690 0.690 srgb} bind def
+/col17 {0.000 0.820 0.820 srgb} bind def
+/col18 {0.560 0.000 0.000 srgb} bind def
+/col19 {0.690 0.000 0.000 srgb} bind def
+/col20 {0.820 0.000 0.000 srgb} bind def
+/col21 {0.560 0.000 0.560 srgb} bind def
+/col22 {0.690 0.000 0.690 srgb} bind def
+/col23 {0.820 0.000 0.820 srgb} bind def
+/col24 {0.500 0.190 0.000 srgb} bind def
+/col25 {0.630 0.250 0.000 srgb} bind def
+/col26 {0.750 0.380 0.000 srgb} bind def
+/col27 {1.000 0.500 0.500 srgb} bind def
+/col28 {1.000 0.630 0.630 srgb} bind def
+/col29 {1.000 0.750 0.750 srgb} bind def
+/col30 {1.000 0.880 0.880 srgb} bind def
+/col31 {1.000 0.840 0.000 srgb} bind def
+/col32 {0.682 0.682 0.682 srgb} bind def
+
+end
+save
+newpath 0 842 moveto 0 0 lineto 595 0 lineto 595 842 lineto closepath clip newpath
+63.5 115.0 translate
+ 90 rotate
+1 -1 scale
+
+/cp {closepath} bind def
+/ef {eofill} bind def
+/gr {grestore} bind def
+/gs {gsave} bind def
+/sa {save} bind def
+/rs {restore} bind def
+/l {lineto} bind def
+/m {moveto} bind def
+/rm {rmoveto} bind def
+/n {newpath} bind def
+/s {stroke} bind def
+/sh {show} bind def
+/slc {setlinecap} bind def
+/slj {setlinejoin} bind def
+/slw {setlinewidth} bind def
+/srgb {setrgbcolor} bind def
+/rot {rotate} bind def
+/sc {scale} bind def
+/sd {setdash} bind def
+/ff {findfont} bind def
+/sf {setfont} bind def
+/scf {scalefont} bind def
+/sw {stringwidth} bind def
+/tr {translate} bind def
+/tnt {dup dup currentrgbcolor
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add srgb}
+  bind def
+/shd {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
+  4 -2 roll mul srgb} bind def
+/$F2psBegin {$F2psDict begin /$F2psEnteredState save def} def
+/$F2psEnd {$F2psEnteredState restore end} def
+
+$F2psBegin
+%%Page: 1 1
+10 setmiterlimit
+ 0.06000 0.06000 sc
+%%Page: 1 1
+%
+% Fig objects follow
+%
+% Polyline
+7.500 slw
+n 2700 1500 m 3300 1500 l 3300 2100 l 2700 2100 l
+ cp gs col7 s gr 
+% Polyline
+n 3300 2100 m 3900 2100 l 3900 2700 l 3300 2700 l
+ cp gs col7 s gr 
+% Polyline
+n 3900 2700 m 4500 2700 l 4500 3300 l 3900 3300 l
+ cp gs col7 s gr 
+/Helvetica ff 360.00 scf sf
+2940 1942 m
+gs 1 -1 sc (I) col0 sh gr
+/Helvetica ff 360.00 scf sf
+3495 2542 m
+gs 1 -1 sc (L) col0 sh gr
+/Helvetica ff 360.00 scf sf
+4140 3142 m
+gs 1 -1 sc (I) col0 sh gr
+% Polyline
+1 slj
+60.000 slw
+n 2700 2700 m
+ 5100 2700 l gs col32 s gr 
+% Polyline
+n 3900 2100 m
+ 2700 2100 l gs col32 s gr 
+% Polyline
+n 3300 1500 m
+ 3300 2700 l gs col32 s gr 
+% Polyline
+n 3900 3900 m
+ 3900 1500 l gs col32 s gr 
+% Polyline
+n 2700 1500 m 7500 1500 l 7500 6300 l 2700 6300 l
+ cp gs col32 s gr 
+% Polyline
+n 5100 1500 m
+ 5100 6300 l gs col32 s gr 
+% Polyline
+n 2700 3900 m
+ 7500 3900 l gs col32 s gr 
+% Polyline
+n 2700 3900 m 3000 3900 l 3900 5100 l 5100 1500 l 6300 5100 l 7200 3900 l
+
+ 7500 3900 l gs col0 s gr 
+% Polyline
+0 slj
+7.500 slw
+n 5100 2025 m 4500 3900 l 5700 3900 l
+ 5100 2025 l  cp gs col18 1.00 shd ef gr gs col18 s gr 
+$F2psEnd
+rs
+showpage
--- /dev/null
+++ waili-19990723/man/Wavelet_dep.eps
@@ -0,0 +1,709 @@
+%!PS-Adobe-2.0
+%%Title: Wavelet_dep.fig
+%%Creator: fig2dev Version 3.2 Patchlevel 3b
+%%CreationDate: Thu May 17 12:30:00 2001
+%%For: debian@etabeta (A C G Mennucc,,,)
+%%Orientation: Portrait
+%%Pages: 2
+%%BoundingBox: 0 0 595 842
+%%BeginSetup
+%%IncludeFeature: *PageSize A4
+%%EndSetup
+%%Magnification: 1.0000
+%%EndComments
+%
+%   
+%    Copyright (C) 1996-1999 Department of Computer Science, K.U.Leuven, Belgium 
+%   
+%    This program is free software; you can redistribute it and/or modify 
+%    it under the terms of the GNU General Public License as published by 
+%    the Free Software Foundation; either version 2 of the License, or 
+%    (at your option) any later version. 
+%   
+%    This program is distributed in the hope that it will be useful, 
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of 
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+%    GNU General Public License for more details. 
+%   
+%    You should have received a copy of the GNU General Public License 
+%    along with this program; if not, write to the Free Software 
+%    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+%   
+%
+/$F2psDict 200 dict def
+$F2psDict begin
+$F2psDict /mtrx matrix put
+/col-1 {0 setgray} bind def
+/col0 {0.000 0.000 0.000 srgb} bind def
+/col1 {0.000 0.000 1.000 srgb} bind def
+/col2 {0.000 1.000 0.000 srgb} bind def
+/col3 {0.000 1.000 1.000 srgb} bind def
+/col4 {1.000 0.000 0.000 srgb} bind def
+/col5 {1.000 0.000 1.000 srgb} bind def
+/col6 {1.000 1.000 0.000 srgb} bind def
+/col7 {1.000 1.000 1.000 srgb} bind def
+/col8 {0.000 0.000 0.560 srgb} bind def
+/col9 {0.000 0.000 0.690 srgb} bind def
+/col10 {0.000 0.000 0.820 srgb} bind def
+/col11 {0.530 0.810 1.000 srgb} bind def
+/col12 {0.000 0.560 0.000 srgb} bind def
+/col13 {0.000 0.690 0.000 srgb} bind def
+/col14 {0.000 0.820 0.000 srgb} bind def
+/col15 {0.000 0.560 0.560 srgb} bind def
+/col16 {0.000 0.690 0.690 srgb} bind def
+/col17 {0.000 0.820 0.820 srgb} bind def
+/col18 {0.560 0.000 0.000 srgb} bind def
+/col19 {0.690 0.000 0.000 srgb} bind def
+/col20 {0.820 0.000 0.000 srgb} bind def
+/col21 {0.560 0.000 0.560 srgb} bind def
+/col22 {0.690 0.000 0.690 srgb} bind def
+/col23 {0.820 0.000 0.820 srgb} bind def
+/col24 {0.500 0.190 0.000 srgb} bind def
+/col25 {0.630 0.250 0.000 srgb} bind def
+/col26 {0.750 0.380 0.000 srgb} bind def
+/col27 {1.000 0.500 0.500 srgb} bind def
+/col28 {1.000 0.630 0.630 srgb} bind def
+/col29 {1.000 0.750 0.750 srgb} bind def
+/col30 {1.000 0.880 0.880 srgb} bind def
+/col31 {1.000 0.840 0.000 srgb} bind def
+
+end
+save
+newpath 0 842 moveto 0 0 lineto 595 0 lineto 595 842 lineto closepath clip newpath
+-130.5 850.5 translate
+1 -1 scale
+
+/cp {closepath} bind def
+/ef {eofill} bind def
+/gr {grestore} bind def
+/gs {gsave} bind def
+/sa {save} bind def
+/rs {restore} bind def
+/l {lineto} bind def
+/m {moveto} bind def
+/rm {rmoveto} bind def
+/n {newpath} bind def
+/s {stroke} bind def
+/sh {show} bind def
+/slc {setlinecap} bind def
+/slj {setlinejoin} bind def
+/slw {setlinewidth} bind def
+/srgb {setrgbcolor} bind def
+/rot {rotate} bind def
+/sc {scale} bind def
+/sd {setdash} bind def
+/ff {findfont} bind def
+/sf {setfont} bind def
+/scf {scalefont} bind def
+/sw {stringwidth} bind def
+/tr {translate} bind def
+/tnt {dup dup currentrgbcolor
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add
+  4 -2 roll dup 1 exch sub 3 -1 roll mul add srgb}
+  bind def
+/shd {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
+  4 -2 roll mul srgb} bind def
+/$F2psBegin {$F2psDict begin /$F2psEnteredState save def} def
+/$F2psEnd {$F2psEnteredState restore end} def
+
+$F2psBegin
+%%Page: 1 1
+10 setmiterlimit
+ 0.06299 0.06299 sc
+%%Page: 1 1
+%
+% Fig objects follow
+%
+% Polyline
+7.500 slw
+n 3170 2666 m 4885 2666 l 4885 3094 l 3170 3094 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+3472 2925 m
+gs 1 -1 sc (Wavelet_Lazy) col-1 sh gr
+% Polyline
+n 3147 4240 m 4862 4240 l 4862 4669 l 3147 4669 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+3277 4500 m
+gs 1 -1 sc (Wavelet_CDF_1_x) col-1 sh gr
+% Polyline
+n 3147 6311 m 4862 6311 l 4862 6739 l 3147 6739 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+3277 6570 m
+gs 1 -1 sc (Wavelet_CDF_2_x) col-1 sh gr
+% Polyline
+n 3147 8426 m 4862 8426 l 4862 8854 l 3147 8854 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+3277 8685 m
+gs 1 -1 sc (Wavelet_CDF_4_x) col-1 sh gr
+% Polyline
+n 3170 9866 m 4885 9866 l 4885 10294 l 3170 10294 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+3262 10125 m
+gs 1 -1 sc (Wavelet_CRF_13_7) col-1 sh gr
+% Polyline
+n 3170 10541 m 4885 10541 l 4885 10969 l 3170 10969 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+3232 10800 m
+gs 1 -1 sc (Wavelet_SWE_13_7) col-1 sh gr
+% Polyline
+n 5915 9055 m 7630 9055 l 7630 9484 l 5915 9484 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 9315 m
+gs 1 -1 sc (Wavelet_CDF_4_6) col-1 sh gr
+% Polyline
+n 5915 8426 m 7630 8426 l 7630 8854 l 5915 8854 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 8685 m
+gs 1 -1 sc (Wavelet_CDF_4_4) col-1 sh gr
+% Polyline
+n 5915 7773 m 7630 7773 l 7630 8201 l 5915 8201 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 8032 m
+gs 1 -1 sc (Wavelet_CDF_4_2) col-1 sh gr
+% Polyline
+n 5915 6963 m 7630 6963 l 7630 7391 l 5915 7391 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 7222 m
+gs 1 -1 sc (Wavelet_CDF_2_6) col-1 sh gr
+% Polyline
+n 5915 6311 m 7630 6311 l 7630 6739 l 5915 6739 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 6570 m
+gs 1 -1 sc (Wavelet_CDF_2_4) col-1 sh gr
+% Polyline
+n 5915 5681 m 7630 5681 l 7630 6109 l 5915 6109 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 5940 m
+gs 1 -1 sc (Wavelet_CDF_2_2) col-1 sh gr
+% Polyline
+n 5915 4893 m 7630 4893 l 7630 5322 l 5915 5322 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 5152 m
+gs 1 -1 sc (Wavelet_CDF_1_5) col-1 sh gr
+% Polyline
+n 5915 4240 m 7630 4240 l 7630 4669 l 5915 4669 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 4500 m
+gs 1 -1 sc (Wavelet_CDF_1_3) col-1 sh gr
+% Polyline
+n 5915 3610 m 7630 3610 l 7630 4039 l 5915 4039 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+6045 3870 m
+gs 1 -1 sc (Wavelet_CDF_1_1) col-1 sh gr
+% Polyline
+n 8705 4039 m 10420 4039 l 10420 3610 l 8705 3610 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 3870 m
+gs 1 -1 sc (LiftCoef_CDF_1_1) col-1 sh gr
+% Polyline
+n 8705 4669 m 10420 4669 l 10420 4240 l 8705 4240 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 4500 m
+gs 1 -1 sc (LiftCoef_CDF_1_3) col-1 sh gr
+% Polyline
+n 8705 5322 m 10420 5322 l 10420 4893 l 8705 4893 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 5152 m
+gs 1 -1 sc (LiftCoef_CDF_1_5) col-1 sh gr
+% Polyline
+n 8705 6109 m 10420 6109 l 10420 5681 l 8705 5681 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 5940 m
+gs 1 -1 sc (LiftCoef_CDF_2_2) col-1 sh gr
+% Polyline
+n 8705 6739 m 10420 6739 l 10420 6311 l 8705 6311 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 6570 m
+gs 1 -1 sc (LiftCoef_CDF_2_4) col-1 sh gr
+% Polyline
+n 8705 7391 m 10420 7391 l 10420 6963 l 8705 6963 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 7222 m
+gs 1 -1 sc (LiftCoef_CDF_2_6) col-1 sh gr
+% Polyline
+n 8705 8201 m 10420 8201 l 10420 7773 l 8705 7773 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 8032 m
+gs 1 -1 sc (LiftCoef_CDF_4_2) col-1 sh gr
+% Polyline
+n 8705 8854 m 10420 8854 l 10420 8426 l 8705 8426 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 8685 m
+gs 1 -1 sc (LiftCoef_CDF_4_4) col-1 sh gr
+% Polyline
+n 8705 9484 m 10420 9484 l 10420 9055 l 8705 9055 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+8842 9315 m
+gs 1 -1 sc (LiftCoef_CDF_4_6) col-1 sh gr
+% Polyline
+n 11495 8854 m 13210 8854 l 13210 8426 l 11495 8426 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+11632 8685 m
+gs 1 -1 sc (LiftCoef_CDF_4_x) col-1 sh gr
+% Polyline
+n 11495 6739 m 13210 6739 l 13210 6311 l 11495 6311 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+11632 6570 m
+gs 1 -1 sc (LiftCoef_CDF_2_x) col-1 sh gr
+% Polyline
+n 11495 4669 m 13210 4669 l 13210 4240 l 11495 4240 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+11632 4500 m
+gs 1 -1 sc (LiftCoef_CDF_1_x) col-1 sh gr
+% Polyline
+n 380 6310 m 2094 6310 l 2094 6739 l 380 6739 l
+ cp gs col-1 s gr 
+/Times-Roman ff 180.00 scf sf
+915 6592 m
+gs 1 -1 sc (Wavelet) col-1 sh gr
+% Polyline
+gs  clippath
+7632 4428 m 7632 4485 l 7777 4485 l 7663 4457 l 7777 4428 l cp
+eoclip
+n 8718 4457 m
+ 7647 4457 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 4428 m 7663 4457 l 7777 4485 l 7777 4428 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 6500 m 7632 6557 l 7777 6557 l 7663 6529 l 7777 6500 l cp
+eoclip
+n 8718 6529 m
+ 7647 6529 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 6500 m 7663 6529 l 7777 6557 l 7777 6500 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 7143 m 7632 7200 l 7777 7200 l 7663 7172 l 7777 7143 l cp
+eoclip
+n 8718 7172 m
+ 7647 7172 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 7143 m 7663 7172 l 7777 7200 l 7777 7143 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 5857 m 7632 5914 l 7777 5914 l 7663 5886 l 7777 5857 l cp
+eoclip
+n 8718 5886 m
+ 7647 5886 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 5857 m 7663 5886 l 7777 5914 l 7777 5857 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 5071 m 7632 5128 l 7777 5128 l 7663 5100 l 7777 5071 l cp
+eoclip
+n 8718 5100 m
+ 7647 5100 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 5071 m 7663 5100 l 7777 5128 l 7777 5071 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 3785 m 7632 3842 l 7777 3842 l 7663 3814 l 7777 3785 l cp
+eoclip
+n 8718 3814 m
+ 7647 3814 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 3785 m 7663 3814 l 7777 3842 l 7777 3785 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5930 5132 m 5959 5083 l 5834 5008 l 5918 5092 l 5805 5057 l cp
+eoclip
+n 4861 4457 m
+ 5932 5100 l gs col-1 s gr gr
+
+% arrowhead
+n 5805 5057 m 5918 5092 l 5834 5008 l 5805 5057 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5959 3830 m 5930 3781 l 5805 3856 l 5918 3822 l 5834 3905 l cp
+eoclip
+n 4861 4457 m
+ 5932 3814 l gs col-1 s gr gr
+
+% arrowhead
+n 5834 3905 m 5918 3822 l 5805 3856 l 5834 3905 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5947 4485 m 5947 4428 l 5801 4428 l 5916 4457 l 5801 4485 l cp
+eoclip
+n 4861 4457 m
+ 5932 4457 l gs col-1 s gr gr
+
+% arrowhead
+n 5801 4485 m 5916 4457 l 5801 4428 l 5801 4485 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5930 7204 m 5959 7155 l 5834 7080 l 5918 7164 l 5805 7129 l cp
+eoclip
+n 4861 6529 m
+ 5932 7172 l gs col-1 s gr gr
+
+% arrowhead
+n 5805 7129 m 5918 7164 l 5834 7080 l 5805 7129 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5959 5902 m 5930 5853 l 5805 5928 l 5918 5894 l 5834 5977 l cp
+eoclip
+n 4861 6529 m
+ 5932 5886 l gs col-1 s gr gr
+
+% arrowhead
+n 5834 5977 m 5918 5894 l 5805 5928 l 5834 5977 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5947 6557 m 5947 6500 l 5801 6500 l 5916 6529 l 5801 6557 l cp
+eoclip
+n 4861 6529 m
+ 5932 6529 l gs col-1 s gr gr
+
+% arrowhead
+n 5801 6557 m 5916 6529 l 5801 6500 l 5801 6557 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10405 7155 m 10434 7204 l 10559 7129 l 10447 7164 l 10530 7080 l cp
+eoclip
+n 11504 6529 m
+ 10433 7172 l gs col-1 s gr gr
+
+% arrowhead
+n 10530 7080 m 10447 7164 l 10559 7129 l 10530 7080 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10434 5853 m 10405 5902 l 10530 5977 l 10447 5894 l 10559 5928 l cp
+eoclip
+n 11504 6529 m
+ 10433 5886 l gs col-1 s gr gr
+
+% arrowhead
+n 10559 5928 m 10447 5894 l 10530 5977 l 10559 5928 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10418 6500 m 10418 6557 l 10563 6557 l 10449 6529 l 10563 6500 l cp
+eoclip
+n 11504 6529 m
+ 10433 6529 l gs col-1 s gr gr
+
+% arrowhead
+n 10563 6500 m 10449 6529 l 10563 6557 l 10563 6500 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10405 5083 m 10434 5132 l 10559 5057 l 10447 5092 l 10530 5008 l cp
+eoclip
+n 11504 4457 m
+ 10433 5100 l gs col-1 s gr gr
+
+% arrowhead
+n 10530 5008 m 10447 5092 l 10559 5057 l 10530 5008 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10434 3781 m 10405 3830 l 10530 3905 l 10447 3822 l 10559 3856 l cp
+eoclip
+n 11504 4457 m
+ 10433 3814 l gs col-1 s gr gr
+
+% arrowhead
+n 10559 3856 m 10447 3822 l 10530 3905 l 10559 3856 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10418 4428 m 10418 4485 l 10563 4485 l 10449 4457 l 10563 4428 l cp
+eoclip
+n 11504 4457 m
+ 10433 4457 l gs col-1 s gr gr
+
+% arrowhead
+n 10563 4428 m 10449 4457 l 10563 4485 l 10563 4428 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 8600 m 7632 8657 l 7777 8657 l 7663 8629 l 7777 8600 l cp
+eoclip
+n 8718 8629 m
+ 7647 8629 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 8600 m 7663 8629 l 7777 8657 l 7777 8600 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 9243 m 7632 9300 l 7777 9300 l 7663 9272 l 7777 9243 l cp
+eoclip
+n 8718 9272 m
+ 7647 9272 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 9243 m 7663 9272 l 7777 9300 l 7777 9243 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+7632 7957 m 7632 8014 l 7777 8014 l 7663 7986 l 7777 7957 l cp
+eoclip
+n 8718 7986 m
+ 7647 7986 l gs col-1 s gr gr
+
+% arrowhead
+n 7777 7957 m 7663 7986 l 7777 8014 l 7777 7957 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5930 9304 m 5959 9255 l 5834 9180 l 5918 9264 l 5805 9229 l cp
+eoclip
+n 4861 8629 m
+ 5932 9272 l gs col-1 s gr gr
+
+% arrowhead
+n 5805 9229 m 5918 9264 l 5834 9180 l 5805 9229 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5959 8002 m 5930 7953 l 5805 8028 l 5918 7994 l 5834 8077 l cp
+eoclip
+n 4861 8629 m
+ 5932 7986 l gs col-1 s gr gr
+
+% arrowhead
+n 5834 8077 m 5918 7994 l 5805 8028 l 5834 8077 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+5947 8657 m 5947 8600 l 5801 8600 l 5916 8629 l 5801 8657 l cp
+eoclip
+n 4861 8629 m
+ 5932 8629 l gs col-1 s gr gr
+
+% arrowhead
+n 5801 8657 m 5916 8629 l 5801 8600 l 5801 8657 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10405 9255 m 10434 9304 l 10559 9229 l 10447 9264 l 10530 9180 l cp
+eoclip
+n 11504 8629 m
+ 10433 9272 l gs col-1 s gr gr
+
+% arrowhead
+n 10530 9180 m 10447 9264 l 10559 9229 l 10530 9180 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10434 7953 m 10405 8002 l 10530 8077 l 10447 7994 l 10559 8028 l cp
+eoclip
+n 11504 8629 m
+ 10433 7986 l gs col-1 s gr gr
+
+% arrowhead
+n 10559 8028 m 10447 7994 l 10530 8077 l 10559 8028 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+gs  clippath
+10418 8600 m 10418 8657 l 10563 8657 l 10449 8629 l 10563 8600 l cp
+eoclip
+n 11504 8629 m
+ 10433 8629 l gs col-1 s gr gr
+
+% arrowhead
+n 10563 8600 m 10449 8629 l 10563 8657 l 10563 8600 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+3165 6553 m 3165 6496 l 3019 6496 l 3134 6525 l 3019 6553 l cp
+eoclip
+n 2074 6522 m 2076 6522 l 2079 6522 l 2086 6522 l 2097 6522 l 2111 6522 l
+ 2129 6522 l 2151 6522 l 2176 6523 l 2204 6523 l 2235 6523 l
+ 2267 6523 l 2300 6523 l 2335 6523 l 2371 6524 l 2407 6524 l
+ 2445 6524 l 2484 6524 l 2524 6524 l 2566 6524 l 2609 6525 l
+ 2654 6525 l 2700 6525 l 2745 6525 l 2797 6525 l 2845 6525 l
+ 2887 6525 l 2924 6525 l 2957 6525 l 2986 6525 l 3011 6525 l
+ 3034 6525 l 3055 6525 l 3074 6525 l 3090 6525 l 3105 6525 l
+ 3118 6525 l 3128 6525 l 3137 6525 l
+ 3150 6525 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 3019 6553 m 3134 6525 l 3019 6496 l 3019 6553 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+3129 4479 m 3182 4458 l 3128 4323 l 3144 4440 l 3075 4344 l cp
+eoclip
+n 2074 6522 m 2074 6521 l 2075 6519 l 2076 6515 l 2078 6509 l 2081 6500 l
+ 2084 6488 l 2089 6473 l 2095 6455 l 2101 6434 l 2108 6409 l
+ 2117 6383 l 2126 6353 l 2136 6322 l 2147 6288 l 2158 6253 l
+ 2170 6216 l 2182 6179 l 2195 6139 l 2208 6099 l 2222 6058 l
+ 2237 6016 l 2252 5973 l 2267 5928 l 2284 5883 l 2301 5835 l
+ 2319 5787 l 2339 5736 l 2359 5684 l 2380 5630 l 2402 5574 l
+ 2426 5517 l 2450 5458 l 2475 5400 l 2503 5335 l 2531 5273 l
+ 2557 5215 l 2581 5163 l 2603 5115 l 2623 5074 l 2640 5038 l
+ 2655 5007 l 2667 4981 l 2678 4958 l 2687 4940 l 2695 4924 l
+ 2702 4910 l 2709 4898 l 2715 4886 l 2721 4876 l 2727 4865 l
+ 2733 4853 l 2741 4839 l 2750 4824 l 2760 4807 l 2771 4787 l
+ 2785 4765 l 2800 4739 l 2817 4711 l 2836 4680 l 2857 4647 l
+ 2879 4612 l 2902 4578 l 2925 4545 l 2944 4519 l 2963 4495 l
+ 2980 4474 l 2996 4457 l 3010 4442 l 3024 4429 l 3036 4419 l
+ 3047 4411 l 3058 4405 l 3067 4401 l 3076 4398 l 3083 4397 l
+ 3091 4397 l 3098 4398 l 3104 4400 l 3110 4402 l 3115 4406 l
+ 3120 4410 l 3124 4414 l 3128 4419 l 3132 4423 l 3136 4428 l
+ 3139 4433 l 3141 4437 l 3143 4441 l 3145 4445 l 3147 4448 l
+ 3148 4450 l
+ 3150 4455 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 3075 4344 m 3144 4440 l 3128 4323 l 3075 4344 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+3182 8591 m 3129 8570 l 3075 8705 l 3144 8610 l 3128 8726 l cp
+eoclip
+n 2074 6528 m 2074 6529 l 2075 6531 l 2076 6535 l 2078 6541 l 2081 6550 l
+ 2084 6562 l 2089 6577 l 2095 6595 l 2101 6616 l 2108 6641 l
+ 2117 6667 l 2126 6697 l 2136 6728 l 2147 6762 l 2158 6797 l
+ 2170 6834 l 2182 6871 l 2195 6911 l 2208 6951 l 2222 6992 l
+ 2237 7034 l 2252 7077 l 2267 7122 l 2284 7167 l 2301 7215 l
+ 2319 7263 l 2339 7314 l 2359 7366 l 2380 7420 l 2402 7476 l
+ 2426 7533 l 2450 7592 l 2475 7650 l 2503 7715 l 2531 7777 l
+ 2557 7835 l 2581 7887 l 2603 7935 l 2623 7976 l 2640 8012 l
+ 2655 8043 l 2667 8069 l 2678 8092 l 2687 8110 l 2695 8126 l
+ 2702 8140 l 2709 8152 l 2715 8164 l 2721 8174 l 2727 8185 l
+ 2733 8197 l 2741 8211 l 2750 8226 l 2760 8243 l 2771 8263 l
+ 2785 8285 l 2800 8311 l 2817 8339 l 2836 8370 l 2857 8403 l
+ 2879 8438 l 2902 8472 l 2925 8505 l 2944 8531 l 2963 8555 l
+ 2980 8576 l 2996 8593 l 3010 8608 l 3024 8621 l 3036 8631 l
+ 3047 8639 l 3058 8645 l 3067 8649 l 3076 8652 l 3083 8653 l
+ 3091 8653 l 3098 8652 l 3104 8650 l 3110 8648 l 3115 8644 l
+ 3120 8640 l 3124 8636 l 3128 8631 l 3132 8627 l 3136 8622 l
+ 3139 8617 l 3141 8613 l 3143 8609 l 3145 8605 l 3147 8602 l
+ 3148 8600 l
+ 3150 8595 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 3128 8726 m 3144 8610 l 3075 8705 l 3128 8726 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+3127 2993 m 3181 2975 l 3135 2837 l 3145 2955 l 3081 2855 l cp
+eoclip
+n 2070 6525 m 2070 6524 l 2070 6522 l 2071 6518 l 2072 6512 l 2073 6504 l
+ 2075 6492 l 2077 6477 l 2079 6459 l 2082 6437 l 2086 6411 l
+ 2090 6382 l 2095 6349 l 2100 6313 l 2106 6273 l 2112 6230 l
+ 2119 6184 l 2126 6135 l 2133 6084 l 2141 6031 l 2149 5976 l
+ 2158 5919 l 2167 5861 l 2176 5802 l 2186 5741 l 2196 5680 l
+ 2206 5618 l 2216 5555 l 2227 5491 l 2238 5427 l 2249 5362 l
+ 2260 5295 l 2272 5229 l 2285 5161 l 2298 5092 l 2311 5022 l
+ 2325 4951 l 2339 4879 l 2354 4806 l 2370 4732 l 2386 4656 l
+ 2403 4580 l 2420 4503 l 2438 4427 l 2456 4350 l 2475 4275 l
+ 2500 4177 l 2525 4084 l 2548 3998 l 2570 3920 l 2591 3850 l
+ 2609 3789 l 2626 3735 l 2641 3690 l 2654 3651 l 2666 3619 l
+ 2676 3593 l 2684 3572 l 2692 3555 l 2698 3541 l 2704 3530 l
+ 2710 3521 l 2715 3514 l 2720 3507 l 2725 3499 l 2731 3491 l
+ 2737 3482 l 2745 3471 l 2753 3457 l 2762 3441 l 2772 3421 l
+ 2784 3397 l 2797 3370 l 2812 3339 l 2828 3305 l 2846 3268 l
+ 2865 3228 l 2885 3186 l 2905 3145 l 2925 3105 l 2942 3074 l
+ 2957 3045 l 2972 3020 l 2986 2997 l 3000 2977 l 3012 2960 l
+ 3023 2945 l 3034 2932 l 3044 2922 l 3053 2914 l 3061 2907 l
+ 3069 2902 l 3076 2899 l 3083 2897 l 3089 2896 l 3095 2897 l
+ 3100 2898 l 3106 2900 l 3110 2903 l 3115 2907 l 3119 2911 l
+ 3123 2915 l 3127 2920 l 3130 2925 l 3133 2931 l 3136 2936 l
+ 3138 2941 l 3141 2946 l 3143 2950 l 3144 2954 l 3146 2958 l
+ 3147 2961 l 3148 2964 l
+ 3150 2970 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 3081 2855 m 3145 2955 l 3135 2837 l 3081 2855 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+3181 10074 m 3127 10056 l 3081 10194 l 3145 10095 l 3135 10212 l cp
+eoclip
+n 2070 6525 m 2070 6526 l 2070 6528 l 2071 6532 l 2072 6538 l 2073 6546 l
+ 2075 6558 l 2077 6573 l 2079 6591 l 2082 6613 l 2086 6639 l
+ 2090 6668 l 2095 6701 l 2100 6737 l 2106 6777 l 2112 6820 l
+ 2119 6866 l 2126 6915 l 2133 6966 l 2141 7019 l 2149 7074 l
+ 2158 7131 l 2167 7189 l 2176 7248 l 2186 7309 l 2196 7370 l
+ 2206 7432 l 2216 7495 l 2227 7559 l 2238 7623 l 2249 7688 l
+ 2260 7755 l 2272 7821 l 2285 7889 l 2298 7958 l 2311 8028 l
+ 2325 8099 l 2339 8171 l 2354 8244 l 2370 8318 l 2386 8394 l
+ 2403 8470 l 2420 8547 l 2438 8623 l 2456 8700 l 2475 8775 l
+ 2500 8873 l 2525 8966 l 2548 9052 l 2570 9130 l 2591 9200 l
+ 2609 9261 l 2626 9315 l 2641 9360 l 2654 9399 l 2666 9431 l
+ 2676 9457 l 2684 9478 l 2692 9495 l 2698 9509 l 2704 9520 l
+ 2710 9529 l 2715 9536 l 2720 9543 l 2725 9551 l 2731 9559 l
+ 2737 9568 l 2745 9579 l 2753 9593 l 2762 9609 l 2772 9629 l
+ 2784 9653 l 2797 9680 l 2812 9711 l 2828 9745 l 2846 9782 l
+ 2865 9822 l 2885 9864 l 2905 9905 l 2925 9945 l 2942 9976 l
+ 2957 10005 l 2972 10030 l 2986 10053 l 3000 10073 l 3012 10090 l
+ 3023 10105 l 3034 10118 l 3044 10128 l 3053 10136 l 3061 10143 l
+ 3069 10148 l 3076 10151 l 3083 10153 l 3089 10154 l 3095 10153 l
+ 3100 10152 l 3106 10150 l 3110 10147 l 3115 10143 l 3119 10139 l
+ 3123 10135 l 3127 10130 l 3130 10125 l 3133 10119 l 3136 10114 l
+ 3138 10109 l 3141 10104 l 3143 10100 l 3144 10096 l 3146 10092 l
+ 3147 10089 l 3148 10086 l
+ 3150 10080 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 3135 10212 m 3145 10095 l 3081 10194 l 3135 10212 l  cp gs 0.00 setgray ef gr  col-1 s
+% Polyline
+2 slj
+gs  clippath
+3189 10748 m 3135 10730 l 3089 10868 l 3153 10769 l 3143 10886 l cp
+eoclip
+n 2070 6525 m 2070 6526 l 2070 6527 l 2071 6531 l 2071 6536 l 2072 6543 l
+ 2073 6553 l 2074 6566 l 2076 6583 l 2078 6602 l 2081 6625 l
+ 2083 6652 l 2087 6682 l 2090 6716 l 2094 6753 l 2099 6794 l
+ 2104 6838 l 2109 6886 l 2114 6936 l 2120 6989 l 2127 7045 l
+ 2133 7103 l 2140 7163 l 2147 7225 l 2154 7289 l 2162 7354 l
+ 2170 7420 l 2178 7488 l 2186 7556 l 2195 7625 l 2203 7695 l
+ 2212 7766 l 2221 7837 l 2230 7909 l 2240 7981 l 2250 8053 l
+ 2260 8126 l 2270 8200 l 2280 8274 l 2291 8349 l 2302 8425 l
+ 2314 8501 l 2326 8578 l 2338 8655 l 2351 8734 l 2364 8813 l
+ 2377 8892 l 2391 8972 l 2406 9053 l 2421 9133 l 2436 9214 l
+ 2451 9293 l 2467 9372 l 2483 9449 l 2508 9566 l 2533 9675 l
+ 2557 9774 l 2579 9864 l 2600 9942 l 2618 10010 l 2635 10068 l
+ 2650 10117 l 2663 10156 l 2675 10188 l 2685 10213 l 2693 10231 l
+ 2701 10245 l 2707 10254 l 2713 10260 l 2719 10264 l 2724 10266 l
+ 2729 10268 l 2734 10270 l 2740 10272 l 2746 10276 l 2753 10282 l
+ 2761 10291 l 2770 10302 l 2780 10318 l 2792 10338 l 2805 10362 l
+ 2820 10390 l 2836 10422 l 2854 10458 l 2873 10497 l 2893 10538 l
+ 2913 10579 l 2933 10619 l 2950 10650 l 2965 10679 l 2980 10704 l
+ 2994 10727 l 3008 10747 l 3020 10764 l 3031 10779 l 3042 10792 l
+ 3052 10802 l 3061 10810 l 3069 10817 l 3077 10822 l 3084 10825 l
+ 3091 10827 l 3097 10828 l 3103 10827 l 3108 10826 l 3114 10824 l
+ 3118 10821 l 3123 10817 l 3127 10813 l 3131 10809 l 3135 10804 l
+ 3138 10799 l 3141 10793 l 3144 10788 l 3146 10783 l 3149 10778 l
+ 3151 10774 l 3152 10770 l 3154 10766 l 3155 10763 l 3156 10760 l
+
+ 3158 10754 l gs col-1 s gr gr
+
+% arrowhead
+0 slj
+n 3143 10886 m 3153 10769 l 3089 10868 l 3143 10886 l  cp gs 0.00 setgray ef gr  col-1 s
+$F2psEnd
+rs
+showpage
